In [1]:
import pandas as pd
import numpy as np

In [2]:
df = pd.read_csv('G:\내 드라이브\web_crawling\linkareer.csv')

In [3]:
df

Company  \
0                        현대엔지니어링 / 자산관리 자산설비 / 2021 상반기   
1                              삼성생명 / 지점사무지원 / 2021 상반기   
2                     현대엠엔소프트 / 영상처리 및 컴퓨터비전 / 2021 상반기   
3     네이버파이낸셜 / [NAVER FINANCIAL] 하계 SW개발자 인턴십 / 202...   
4             한국지능정보사회지능원 / 데이터 분석 및 운영 청년인턴 / 2021 상반기   
...                                                 ...   
9874                             웅진코웨이 / 마케팅 / 2013 하반기   
9875                            현대자동차 / 영업관리 / 2013 상반기   
9876                               심팩 / 용접관리 / 2013 상반기   
9877                         한국전력 / 인턴[송,배전] / 2013 상반기   
9878                         한국전력 / 인턴[송,배전] / 2013 상반기   

                                              UserInfos  \
0     한양대 ERICA / 기계공학과 / 학점 3.78/4.5 / 토익: 855, 토익스...   
1     지방사립 / 경제학 / 학점 4.15/4.5 / 토익: 720, 토익스피킹: 140...   
2     한서삼 / 컴퓨터공학과 / 학점 4.44 / 토익: 765, 토익스피킹: , 오픽:...   
3     인하대 / 컴퓨터공학과 / 학점 3.36 / 토익: , 토익스피킹: , 오픽: IM...   
4     경기대학교 / 응용통계학과 / 학점 3.8 / 토익: 705, 토익스피킹: , 오픽...   
...                                                 ...   
9874                 부산경상대학교 / 광고마케팅과 / 학점 3.8 / 토익 850   
9875  상명대학교 / 국제통상학 / 학점 4.3 / 토익 900, HSK 6급 / 해외인턴...   
9876                             지방전문대 / 공업화학 / 학점 3.06   
9877      지거국 / 전자공학 / 학점 3.7 / 토익 900대 / 음악봉사활동, 멘토링 등   
9878        지거국 / 전자공학 / 학점 3.7 / 토익 900 / 음악봉사활동, 멘토링등   

                                                Content  \
0     본인이 회사를 선택할때의 기준은 무엇이며, 왜 현대엔지니어링이 그 기준에 적합한지를...   
1     [금융 백종원] 백종원은 여러 매체를 통해 요리에 관한 각종 정보와 비결을 공유하며...   
2     현대엠엔소프트에 지원한 이유가 무엇인가요? (회사 및 직무에 대한 흥미와 커리어 계...   
3     본인이 이수한 SW 관련 전공 중, 가장 즐겁게 배웠던 수업 혹은 가장 좋은 성적을...   
4     1. 해당 지원분야 지원 동기와 해당 사업 참가 후 계획을 기술해주십시오(500자 ...   
...                                                 ...   
9874  1. 지원동기\n\n인재와 기업이 함께 성장할 수 있는 곳, 그곳이 바로 웅진코웨이...   
9875  1. 현대자동차에 지원하게 된 동기 또는 해당 직무에 지원하게 된 동기는 무엇입니까...   
9876  성격소개\n\n대부분의 사람들이 평가하는 첫인상은 소심하고 내성적인 성격일것이라 생...   
9877  [질문 1] 취업이라는 목표 이외에 인생의 큰 목표를 세운 경험, 그 목표를 실현하...   
9878  한전 자가소개서 양식 500자 12줄 이내\n\n[질문 1] 취업이라는 목표 이외에...   

                                                    URL  
0     https://linkareer.com/cover-letter/25204?page=...  
1     https://linkareer.com/cover-letter/25579?page=...  
2     https://linkareer.com/cover-letter/28660?page=...  
3     https://linkareer.com/cover-letter/28662?page=...  
4     https://linkareer.com/cover-letter/28755?page=...  
...                                                 ...  
9874  https://linkareer.com/cover-letter/13306?page=...  
9875  https://linkareer.com/cover-letter/20654?page=...  
9876  https://linkareer.com/cover-letter/22028?page=...  
9877  https://linkareer.com/cover-letter/22089?page=...  
9878  https://linkareer.com/cover-letter/17456?page=...  

[9879 rows x 4 columns]

### null데이터 확인

In [4]:
df.isnull().sum()

Company        0
UserInfos    370
Content        0
URL            0
dtype: int64

In [5]:
df[df['UserInfos'].isnull() == True].URL

1777    https://linkareer.com/cover-letter/20460?page=...
1782    https://linkareer.com/cover-letter/19220?page=...
1915    https://linkareer.com/cover-letter/21139?page=...
2074    https://linkareer.com/cover-letter/23140?page=...
2102    https://linkareer.com/cover-letter/22918?page=...
                              ...                        
9719    https://linkareer.com/cover-letter/12952?page=...
9722    https://linkareer.com/cover-letter/11956?page=...
9728    https://linkareer.com/cover-letter/22275?page=...
9746    https://linkareer.com/cover-letter/15398?page=...
9860    https://linkareer.com/cover-letter/18264?page=...
Name: URL, Length: 370, dtype: object

## [1] Company확인

### [1-1] Company 정보가 3개 이상

* Company 정보는 원래 [회사]/[직군]/[날짜] 형식이어야 되는데 직군을 길게 적을 경우 ("/") split 시 3개 이상 됨
* 이런 데이터 확인해서 변경하기

In [6]:
Company = df.Company.tolist()

In [7]:
Company = [com.split('/') for com in Company]
Company = [[c.strip() for c in com] for com in Company]

three_over = []
over = []
j = 0
for idx, com in enumerate(Company):
    if len(com) > 3: 
        three_over.append([idx, com, j])
        j += 1
        if len(com) > 4:
            over.append([idx, com])

In [8]:
len(three_over)

563

In [9]:
len(over)

50

In [47]:
three_over

[[5, ['펄어비스', '마케터', '게임마케터', '2021 상반기'], 0],
 [8,
  ['네이버',
   '[지식베이스] 네이버 영화 서비스 개발 신입',
   '경력사원 모집 - (신입) 네이버 영화 서비스 개발',
   '2021 상반기'],
  1],
 [9, ['한국연구재단', 'IT', '전산', '2021 상반기'], 2],
 [24, ['케이티스카이라이프', '영업', '마케팅', 'CS', '2021 상반기'], 3],
 [25,
  ['네이버',
   '[Search] 네이버',
   'LINE 지역검색 및 추천 서비스 개발 신입',
   '경력 모집 - (신입)Back-end',
   'System 엔지니어',
   '2021 상반기'],
  4],
 [26, ['LG에너지솔루션', '연구소', '개발센터 - Cell', '2021 상반기'], 5],
 [47, ['조인스중앙', 'IT 부문 개발', '운영', '2020 하반기'], 6],
 [84, ['LG이노텍', 'H', 'W개발', '2020 하반기'], 7],
 [101, ['아이티포워드', '웹(JAVA', 'JSP) 개발', '2020 하반기'], 8],
 [130, ['JTBC스튜디오', '전략기획', '경영지원', '2020 하반기'], 9],
 [136, ['조인스중앙', 'IT 부문 개발', '운영', '2020 하반기'], 10],
 [182, ['LG CNS', 'LG CNS 스마트팩토리', '스마트물류', '2020 하반기'], 11],
 [209, ['SK주식회사 C&C', 'Data Analytics', 'Engineering', '2020 하반기'], 12],
 [218, ['CJ제일제당', 'Food Sales', '서울', '수도권', '2020 하반기'], 13],
 [231, ['모바일리더', '인지소프트', 'IT', '2020 하반기'], 14],
 [239, ['SK주식회사 C&C', 'Data Analytics', 'Engineering

In [10]:
CompanyName = []
CompanyCareer = []
CompanyDate = []

for com in Company:
    if len(com) <= 3:
        CompanyName.append(com[0])
        CompanyCareer.append(com[1])
        CompanyDate.append(com[2])
    else:
        CompanyName.append(com[0])
        CompanyDate.append(com[-1])
        CompanyCareer.append(" ".join(com[1:-1]))

In [11]:
df['CompanyName'] = CompanyName
df['CompanyCareer'] = CompanyCareer
df['CompanyDate'] = CompanyDate

In [12]:
df.loc[9]

Company                                  한국연구재단 / IT/전산 / 2021 상반기
UserInfos        지거국 / 컴퓨터 / 학점 3.9/4.3 / 토익: 765, 토익스피킹: , 오픽:...
Content          1. 우리 재단 인턴으로 지원한 가장 큰 동기 및 인턴기간 동안 이루고자하는 목표에...
URL              https://linkareer.com/cover-letter/28761?page=...
CompanyName                                                 한국연구재단
CompanyCareer                                                IT 전산
CompanyDate                                               2021 상반기
Name: 9, dtype: object

In [13]:
df = df.drop('Company', axis=1)

In [14]:
df

UserInfos  \
0     한양대 ERICA / 기계공학과 / 학점 3.78/4.5 / 토익: 855, 토익스...   
1     지방사립 / 경제학 / 학점 4.15/4.5 / 토익: 720, 토익스피킹: 140...   
2     한서삼 / 컴퓨터공학과 / 학점 4.44 / 토익: 765, 토익스피킹: , 오픽:...   
3     인하대 / 컴퓨터공학과 / 학점 3.36 / 토익: , 토익스피킹: , 오픽: IM...   
4     경기대학교 / 응용통계학과 / 학점 3.8 / 토익: 705, 토익스피킹: , 오픽...   
...                                                 ...   
9874                 부산경상대학교 / 광고마케팅과 / 학점 3.8 / 토익 850   
9875  상명대학교 / 국제통상학 / 학점 4.3 / 토익 900, HSK 6급 / 해외인턴...   
9876                             지방전문대 / 공업화학 / 학점 3.06   
9877      지거국 / 전자공학 / 학점 3.7 / 토익 900대 / 음악봉사활동, 멘토링 등   
9878        지거국 / 전자공학 / 학점 3.7 / 토익 900 / 음악봉사활동, 멘토링등   

                                                Content  \
0     본인이 회사를 선택할때의 기준은 무엇이며, 왜 현대엔지니어링이 그 기준에 적합한지를...   
1     [금융 백종원] 백종원은 여러 매체를 통해 요리에 관한 각종 정보와 비결을 공유하며...   
2     현대엠엔소프트에 지원한 이유가 무엇인가요? (회사 및 직무에 대한 흥미와 커리어 계...   
3     본인이 이수한 SW 관련 전공 중, 가장 즐겁게 배웠던 수업 혹은 가장 좋은 성적을...   
4     1. 해당 지원분야 지원 동기와 해당 사업 참가 후 계획을 기술해주십시오(500자 ...   
...                                                 ...   
9874  1. 지원동기\n\n인재와 기업이 함께 성장할 수 있는 곳, 그곳이 바로 웅진코웨이...   
9875  1. 현대자동차에 지원하게 된 동기 또는 해당 직무에 지원하게 된 동기는 무엇입니까...   
9876  성격소개\n\n대부분의 사람들이 평가하는 첫인상은 소심하고 내성적인 성격일것이라 생...   
9877  [질문 1] 취업이라는 목표 이외에 인생의 큰 목표를 세운 경험, 그 목표를 실현하...   
9878  한전 자가소개서 양식 500자 12줄 이내\n\n[질문 1] 취업이라는 목표 이외에...   

                                                    URL  CompanyName  \
0     https://linkareer.com/cover-letter/25204?page=...      현대엔지니어링   
1     https://linkareer.com/cover-letter/25579?page=...         삼성생명   
2     https://linkareer.com/cover-letter/28660?page=...      현대엠엔소프트   
3     https://linkareer.com/cover-letter/28662?page=...      네이버파이낸셜   
4     https://linkareer.com/cover-letter/28755?page=...  한국지능정보사회지능원   
...                                                 ...          ...   
9874  https://linkareer.com/cover-letter/13306?page=...        웅진코웨이   
9875  https://linkareer.com/cover-letter/20654?page=...        현대자동차   
9876  https://linkareer.com/cover-letter/22028?page=...           심팩   
9877  https://linkareer.com/cover-letter/22089?page=...         한국전력   
9878  https://linkareer.com/cover-letter/17456?page=...         한국전력   

                       CompanyCareer CompanyDate  
0                          자산관리 자산설비    2021 상반기  
1                             지점사무지원    2021 상반기  
2                       영상처리 및 컴퓨터비전    2021 상반기  
3     [NAVER FINANCIAL] 하계 SW개발자 인턴십    2021 상반기  
4                   데이터 분석 및 운영 청년인턴    2021 상반기  
...                              ...         ...  
9874                             마케팅    2013 하반기  
9875                            영업관리    2013 상반기  
9876                            용접관리    2013 상반기  
9877                        인턴[송,배전]    2013 상반기  
9878                        인턴[송,배전]    2013 상반기  

[9879 rows x 6 columns]

In [15]:
df.isnull().sum()

UserInfos        370
Content            0
URL                0
CompanyName        0
CompanyCareer      0
CompanyDate        0
dtype: int64

In [16]:
df.to_csv('G:\내 드라이브\web_crawling\linkareer_edit1.csv', mode="w", index=False)

### [1-2] 회사 확인

* Company Name => 소문자 모두 대문자로 변경하여 같은 회사 합치기

In [104]:
import re

In [63]:
CompanyNames = set(df.CompanyName.unique().tolist())

In [64]:
len(CompanyNames)

1548

In [102]:
name_list = df.CompanyName.tolist()

In [105]:
name_list = [re.sub("\((.*?)\)", "", name.upper().replace(" ", "").replace("(재)", "").replace("(주)", "").replace("㈜","")) for name in name_list]

In [110]:
df.CompanyName = name_list

In [106]:
CompanyNames = list(set(name_list))

In [107]:
len(CompanyNames)

1323

In [108]:
CompanyNames.sort()

In [109]:
CompanyNames

['11번가',
 'AK유통',
 'AMK',
 'APPLIEDMATERIALSKOREA',
 'ASML',
 'ASML코리아',
 'BC카드',
 'BDKOREA',
 'BGF',
 'BGF리테일',
 'BNK캐피탈',
 'BR코리아',
 'CELLTRION',
 'CGV',
 'CJ',
 'CJ4DPLEX',
 'CJCGV',
 'CJENM',
 'CJ건설',
 'CJ계열사',
 'CJ네트웍스',
 'CJ대한통운',
 'CJ대한통운건설부문',
 'CJ생물자원',
 'CJ시스템즈',
 'CJ오쇼핑',
 'CJ올리브네트웍스',
 'CJ올리브영',
 'CJ이엔엠',
 'CJ제일기획',
 'CJ제일제당',
 'CJ파워캐스트',
 'CJ푸드빌',
 'CJ프레시안',
 'CJ프레시웨이',
 'CJ헬로비전',
 'CJ헬스케어',
 'DB금융투자',
 'DB손해보험',
 'DB하이텍',
 'DGB대구은행',
 'DSR제강',
 'E1',
 'EY한영',
 'G2R',
 'GSE&R',
 'GSEPS',
 'GSSHOP',
 'GS글로벌',
 'GS네오텍',
 'GS동해전력',
 'GS리테일',
 'GS스포츠',
 'GS에너지',
 'GS엠비즈',
 'GS칼텍스',
 'GS홈쇼핑',
 'HMC투자증권',
 'IBK기업은행',
 'IBK투자증권',
 'JTBC스튜디오',
 'JW중외제약',
 'KAC공항서비스',
 'KB국민은행',
 'KB부동산신탁',
 'KB신용정보',
 'KB저축은행',
 'KCC',
 'KDB산업은행',
 'KDI한국개발연구원',
 'KEB하나은행',
 'KGC인삼공사',
 'KG동부제철',
 'KLA',
 'KMAC',
 'KORAIL한국철도공사',
 'KOTITI시험연구원',
 'KOTRA',
 'KPX케미칼',
 'KT',
 'KT&G',
 'KTCS',
 'KTDS',
 'KTESTATE',
 'KTH',
 'KTIS',
 'KTM&S',
 'KTMHOWS',
 'KTM모바일',
 'KTPOWERTEL',
 'KTSKYLIFE',
 'KTTEL

## [2] User 정보

* user 정보가 아예 없는 경우 있음 => 이 경우 그냥 null로 대체

### [2-1] 학점 정보

* 학점은 4가지 형태가 있을 수 있음

1. 학점 3.5/4.5 인 경우 =>  re.search('학점(.?)\d*(\.?\d*)(./)\d*(\.?\d*)(.)/', user_info)
2. 학점 3.5 / 인 경우 => re.search('학점(.?)\d*(\.?\d*)(./)\d*(\.?\d*)(.)/', user_info)
3. 학점 3.5 인 경우 => re.search('학점(.?)\d*(\.?\d*)', user_info)
4. 학점 정보 없음 => if 학점 in user_info

### [2-2] "/"으로 split해서 정보 확인

* 먼저 어떤 특수문자들이 있는지 파악 필요

In [14]:
users = df.UserInfos.tolist()

In [17]:
import re

In [22]:
special = re.compile(r'[^ A-Za-z0-9가-힣+]')
result = set()
for user in users:
    result |= set(special.findall(user))

In [24]:
json_special = dict()

In [25]:
for r in result:
    if r == '/': continue
    json_special[r] = []
    for user in users:
        if r in user: json_special[r].append(user)


In [26]:
json_special.keys()

dict_keys(['-', 'ㄱ', '<', '）', '‘', 'ㅈ', '③', '(', '多', '.', '·', '（', 'ㅗ', '①', ':', '`', '_', '>', ')', '*', '~', 'ㅏ', '害', "'", '新', 'ㄴ', ']', '&', '\\', '有', '\u3000', '[', ',', '?', '外', '②', '中', '無', '=', '’'])

In [27]:
print(len(result))
print(len(json_special.keys()))

41
40


* "-" : pass
* ㄱ : 데이터 전처리 시 'ㄱ' 제거 <어문계열, 경영하ㅏㄱ -> 경영학과로 바꿔주기 이것만>
* < : "<-이건 근데 자소서 내용에 들어가진 않았네요" 제거
* ) : pass
* ‘’ : "지원하는 회사와 분야(직무)에 대한 지원동기를 자유롭게 기술 하세요" 이 부분 제거, 이 문장 말고는 '으로 변경하기
* ㅈ : 제거
* ③, ① : "01.여러분이 선택한 계열사와 해당 직무를 지원한 동기는 무엇인가요?" 이 부분 제거
* ( : pass
* ㅗ : 토ㅗ익 -> 토익으로 변경
* ":" : /말고, :로 구분 지는 부분 -> 한국사검정시험: , 컴퓨터활용능력: , 기타: 정보보안산업기사 or 토익: 765, 토익스피킹: , 오픽: IM1, 기타: / 사회생활 경험: 네이버 인턴 - 일단 pass
* 多, 有, 新 : 이 단어 제거
* 無, 外: 일단 pass 지우지 x
* 中 : 서강대, 성균관대, 한양대로 변경
* ㄴ : -로 변경
* \\ : 제거
* -> : :으로 변경
* \u3000 : 제거
* = : 제거
* ㅏ : "알바 4ㅏ수" -> "알바 4번"
* *: 및으로 변경
* ` : '으로 변경
* ㅗ : "토ㅗ익 870" -> "토익"으로 변경

In [42]:
df[df['UserInfos'].str.contains('(')]['UserInfos'].tolist()

['서울 / 화학 /  토익 920 / 오픽 IH / 토스 160 / 미국 인턴 12개월 봉사 2회 각 12개월, 7개월 01.여러분이 선택한 계열사와 해당 직무를 지원한 동기는 무엇인가요? ① 선택한 계열사가 아니면 안되는 이유, ② 직무에 관심갖게 된 계기, ③ 본인이 직무를 잘 수행할 수 있는 이유(본인의 강점, 노력, 경험 등에 근거)를 반드시 포함하여 구체적으로 작성해주세요. (1000자) 화학을 전공으로서 배우면배울수록 화학이 자연에 모든 해(害)를']

In [65]:
json_special['ㅗ']

['미디어컨텐츠 /  토ㅗ익 870 / 인턴1, 대외활동 2']

## [3] 본문 내용

In [19]:
import re

In [17]:
content = df.Content.tolist()

In [20]:
content = [re.sub("\n+", "\n", c) for c in content]

### [3-1] \n 여러개를 1개로 줄이기

In [39]:
import re

content2 = [re.sub("\n", "@@@", c) for c in content]

### [3-2] 본문을 (질문 - 내용)으로 나누기

#### [3-2-1] 숫자로 시작하는 질문

In [53]:
start_num = []

In [54]:
for idx, con in enumerate(content2):
    if con[0].isdigit(): # 숫자로 시작
        start_num.append([idx, con])

In [55]:
len(start_num)

5151

In [56]:
print([idx for idx, s in start_num][:6])

[4, 6, 9, 10, 11, 15]


In [57]:
# 숫자로 시작하는 경우
# ①, ② : 이런 숫자 모두 1,2,3,로 변경하기
new_contents = []
for idx, con in start_num:
    # [1] \d[. ]?[. ]+(.*?)(\n)+ 사용해서 질문 가져오기
    con = "@@@"+con
    con = con.replace('①', '1 .').replace('②', '2 .').replace('③', '3 .').replace('④', '4 .').replace('⑤', '5 .').replace('⑥', '6 .')
    con = con.replace('⓵', '1 .').replace('⓶', '2 .').replace('⓷', '3 .')
    qa_dict = dict()
    questions = [q[0] for q in re.findall('@@@\d[. )-]?[. )\d]+(.*?)(\.|@@@)', con)]
    answers = re.sub('@@@\d[. )-]?[. )\d]+(.*?)(\.|@@@)', '###############', con).split("###############")[1:]

    for q, a in zip(questions, answers):
        qa_dict[q] = re.sub(' +', ' ', a.replace('@@@',' ')).strip()
    new_contents.append([idx, qa_dict])

In [62]:
new_contents[5]

[15,
 {'본인을 채용하게 되면 포스코케미칼에 영향을 줄 수 있는 긍정적인 측면과 기회에 대해 구체적으로 기술하시오': '[최소:900byte][최대:1800byte]) 낙낙고취(樂樂高就): 일을 즐기면서 하는 사람은 성취도 크다. 저는 3학년 여름방학에 실험실 생활을 시작으로 2차전지를 접하게 됐습니다. 제게 주어진 과제는 바이오매스(대나무)를 기반으로 한 음극재 제조였습니다. 평범히 음극재를 제조하던 중 대나무가 어느 정도까지 흑연화가 됐는지 확인하기 위해 분석 장비를 접하게 됐습니다. XRD를 시작으로 여러 분석 장비를 통해 많은 공을 들인 샘플의 변화를 목격하니 신기했고, 샘플에 대한 애정이 생기기 시작했습니다. 3주 동안 정체를 알 수 없던 이물질 때문에 연구를 진행할 수 없어 고생한 적이 있었습니다. 원인이 볼밀 과정에서 깎여 나온 지르코니아였다는 사실을 발견했을 땐 희열을 느끼기도 했습니다. 그리고 충방전 테스트의 결과를 보기까지 대략 2주의 시간이 소요됐는데, 그 긴 기다림 속에서 인내심을 길렀고, 긴 시간이 걸린 만큼 성취감도 크게 맛봤습니다. 이러한 일들을 겪으면서 연구 활동을 점점 즐기기 시작했고, 제 시간을 샘플에 초점을 맞추어 쓰기 시작했습니다. 제가 그렇게 연구에 몰두할 수 있었던 가장 큰 원동력은 전지에 대한 애정이었습니다. 2차전지 산업은 최근 본격적으로 성장할 단계인 산업이라고 생각됩니다. 이때 전지에 관한 역량과 열정을 갖춘 제가 2차전지 산업에 뛰어든다면, 연구하며 길러낸 저의 역량들과 전지에 대한 애정을 연구지원 직무에 발휘하여 포스코케미칼 셀 제조분석 부문에 있어서 이바지할 수 있는 바가 크다고 생각됩니다.',
  '제한된 자원(시간, 비용, 인력 등)에도 불구하고 도전적인 목표를 달성했던 경험을 구체적으로 기술하시오': '[최소:900byte][최대:1800byte]) 우공이산(愚公移山): 한 가지 일을 끝까지 밀고 나가 성공시킴 저는 20살에 고려대학교 세종캠퍼스 제어계측공학과에 입학했습니다. 원래는 화학을 좋아하여

#### [3-2-2] 처리하고도 남은 부분은 직접 수정하기 : 15개

* 2429번 : 3교내, 대외활동, 회사 등을 통해 협업을 하면서 가장 어려웠던 점은 무엇입니까?교내, 대외활동, 회사 등을 통해 협업을 하면서 가장 어려웠던 점은 무엇입니까?

In [59]:
temp = [idx for idx, c in new_contents if not c]

In [60]:
len(temp)

15

In [78]:
changes = []

In [80]:
for t in temp:
    qa_dict = dict()
    c = content2[t].replace('@@@',' ').strip()
    print(t,":",)
    while True:
        question = input("ENTER Question : ")
        if question == "n": break
        answer = input("ENTER Answer : ")
        qa_dict[question] = answer
    changes.append(qa_dict)
    print()
    print()

2429 : 3교내, 대외활동, 회사 등을 통해 협업을 하면서 가장 어려웠던 점은 무엇입니까? 그 어려운점을 어떻게 해결해 나갔는지에 대해 말씀해주세요 다양한 협력업체 사장님을 만나야 했는데, 처음에는 나이가 어려 사기당하고, 무시당하지 않을까 하는 두려움이 많았습니다. 시장을 돌며 재료 가격을 물어보는 것도 퉁명스럽게 대답하는 사장님들에게 상처도 많이 받았습니다. 그리고 겉모양은 멀쩡한데 제대로 된 검수과정을 거치지 않아 부품이 떨어지는 재료를 더 비싸게 산 적도 있습니다. 그런 아픈 경험을 통해 그 이후로는 급하게 사지 않고 더 시간을 들여 시장을 돌아보고 제품을 찾았습니다.


ENTER Question :  교내, 대외활동, 회사 등을 통해 협업을 하면서 가장 어려웠던 점은 무엇입니까? 그 어려운 점을 어떻게 해결해 나갔는지에 대해 말씀해주세요.
ENTER Answer :  다양한 협력업체 사장님을 만나야 했는데, 처음에는 나이가 어려 사기당하고, 무시당하지 않을까 하는 두려움이 많았습니다. 시장을 돌며 재료 가격을 물어보는 것도 퉁명스럽게 대답하는 사장님들에게 상처도 많이 받았습니다. 그리고 겉모양은 멀쩡한데 제대로 된 검수과정을 거치지 않아 부품이 떨어지는 재료를 더 비싸게 산 적도 있습니다. 그런 아픈 경험을 통해 그 이후로는 급하게 사지 않고 더 시간을 들여 시장을 돌아보고 제품을 찾았습니다.
ENTER Question :  n




2466 : 4번 문항: 혼자 하기 어려운 일에서 다양한 자원 활용, 타인의 협력을 최대한으로 이끌어 내며, Teamwork를 발휘하여 공동의 목표 달성에 기여한 경험에 대해 서술해 주십시오. [효과적인 업무 분담으로 시너지 효과를 내다] ㅇㅇㅇㅇ의 서포터즈의 홍보팀 팀장으로 활동하면서 팀원들과 함께 성공적으로 홍보물을 제작한 경험이 있습니다. 당시 저는 ㅇㅇㅇㅇ주최하는 ㅇㅇㅇㅇ을 홍보하는 영상을 제작하기 위해 팀원을 꾸렸습니다. 당시 홍보팀은 열명이 넘는 인원이 모두 저마다의 의견을 내면서 의사 결정과정이 느렸습니다. 또 많은 인원들 틈에서 소극적으로 참여하는 인원이 발생하면서 불만이 발생할 수도 있는 상황이었습니다. 특히 촬영과 제작 간의 경계가 모호해 영상 제작의 진행이 더딘 상황이었습니다. 저는 우선 일이 효율적으로 배분되도록 팀원 한 명, 한 명에게 개인 메세지를 보내 하고 싶은 일이 무엇인지 들었습니다. 이를 바탕으로 촬영, 제작, 섭외 및 자료 확보 등의 업무를 적당한 인원수로 배분했습니다. 또 홍보팀을 촬영팀, 제작팀, 섭외 및 자료팀으로 나누고 각각 팀장을 정했습니다. 각 팀장과 제가 속한 단체카톡방을 만들어 각 팀의 진행상황을 실시간으로 주고받으며 의견을 조율했습니다. 또한 정기적인 회의 날짜를 정해 진행 속도를 조정했습니다. 팀을 나누고 업무를 할당하니 영상 제작도 원할히 이뤄졌습니다. 각 팀에서 팀원끼리 조율한 의견을 각 팀장이 취합해서 전달해주었습니다. 진행 과정에서 문제가 생겼을 때 역시 팀장을 중심으로 발 빠르게 솔루션을 마련할 수 있었습니다. 그 결과 ㅇㅇㅇㅇ의 형식을 차용해 ㅇㅇㅇㅇ참가자를 모집하는 홍보 영상이 성공적으로 제작됐습니다. 영상은 ㅇㅇㅇㅇ 공식 홈페이지에서 좋은 반응을 얻었습니다.


ENTER Question :  혼자 하기 어려운 일에서 다양한 자원 활용, 타인의 협력을 최대한으로 이끌어 내며, Teamwork를 발휘하여 공동의 목표 달성에 기여한 경험에 대해 서술해 주십시오.
ENTER Answer :  [효과적인 업무 분담으로 시너지 효과를 내다] ㅇㅇㅇㅇ의 서포터즈의 홍보팀 팀장으로 활동하면서 팀원들과 함께 성공적으로 홍보물을 제작한 경험이 있습니다. 당시 저는 ㅇㅇㅇㅇ주최하는 ㅇㅇㅇㅇ을 홍보하는 영상을 제작하기 위해 팀원을 꾸렸습니다. 당시 홍보팀은 열명이 넘는 인원이 모두 저마다의 의견을 내면서 의사 결정과정이 느렸습니다. 또 많은 인원들 틈에서 소극적으로 참여하는 인원이 발생하면서 불만이 발생할 수도 있는 상황이었습니다. 특히 촬영과 제작 간의 경계가 모호해 영상 제작의 진행이 더딘 상황이었습니다. 저는 우선 일이 효율적으로 배분되도록 팀원 한 명, 한 명에게 개인 메세지를 보내 하고 싶은 일이 무엇인지 들었습니다. 이를 바탕으로 촬영, 제작, 섭외 및 자료 확보 등의 업무를 적당한 인원수로 배분했습니다. 또 홍보팀을 촬영팀, 제작팀, 섭외 및 자료팀으로 나누고 각각 팀장을 정했습니다. 각 팀장과 제가 속한 단체카톡방을 만들어 각 팀의 진행상황을 실시간으로 주고받으며 의견을 조율했습니다. 또한 정기적인 회의 날짜를 정해 진행 속도를 조정했습니다. 팀을 나누고 업무를 할당하니 영상 제작도 원할히 이뤄졌습니다. 각 팀에서 팀원끼리 조율한 의견을 각 팀장이 취합해서 전달해주었습니다. 진행 과정에서 문제가 생겼을 때 역시 팀장을 중심으로 발 빠르게 솔루션을 마련할 수 있었습니다. 그 결과 ㅇㅇㅇㅇ의 형식을 차용해 ㅇㅇㅇㅇ참가자를 모집하는 홍보 영상이 성공적으로 제작됐습니다. 영상은 ㅇㅇㅇㅇ 공식 홈페이지에서 좋은 반응을 얻었습니다.
ENTER Question :  n




2512 : 1번 높은 목표 설정 자발적으로 최고 수준의 목표를 세우고 끈질기게 성취한 경험에 대해 서술해 주십시오. (본인이 설정한 목표/목표의 수립 과정/처음에 생각했던 목표 달성 가능성/수행 과정에서 부딪힌 장애물 및 그 때의 감정/목표 달성을 위한 구체적 노력/실제 결과) ‘현실에 안주하지 않고 끊임없이 나를 채워가는 것’이는 저를 데이터분석가라는 꿈으로 꾸준히 달려갈 수 있게 해주었습니다. 준비해야할 것들이 많아 막막했지만 단순히 프로그래밍을 잘하는 것이 아닌 ‘데이터에서 최대의 인사이트를 도출하는 프로젝트 해보기’를 세부 목표로 설정하였습니다. 첫째로 KOCW 강의와 프로그래밍 서적으로 R 언어를 독학하였습니다. 통계학 수업에서 배운 내용과 더불어 이론이 프로그래밍 상에서는 어떻게 구현이 되는지에 집중을 하며 언어 환경을 익히려고 노력하였습니다. 이후에는 데이터마이닝 알고리즘이 실제 데이터에 어떻게 활용이 되는지를 중점으로 공부하였습니다. 하지만 그 과정에서 데이터를 이해하고 산업에 맞는 결과를 도출하기보다는 단순 분석만 하는 스스로를 발견하면서 교수님께서 주의하신 분석기계가 되어가고 있다는 생각이 들었습니다. 같은 문제라도 전공과 사람마다 접근하는 방식이 다르기에 다양한 견해를 논의하고 학습하는 장이 있으면 좋겠다는 생각이 들어 동아리를 설립했습니다. 그곳에서 다양한 전공을 가진 학우들과 함께 프로그래밍과 문제를 다각도로 보는 방식을 배울 수 있었습니다. 처음에 데이터분석을 공부하겠다고 결심했을 때 늦지 않았을까 하는 걱정이 들었습니다. 하지만 한 계단 오른 것에 안주하지 않고 다음 단계를 준비하는 자세로 평소에 어려워했던 마케팅 데이터로 기획까지 제안하는 결과물을 완성할 수 있었습니다.


ENTER Question :  자발적으로 최고 수준의 목표를 세우고 끈질기게 성취한 경험에 대해 서술해 주십시오. (본인이 설정한 목표/목표의 수립 과정/처음에 생각했던 목표 달성 가능성/수행 과정에서 부딪힌 장애물 및 그 때의 감정/목표 달성을 위한 구체적 노력/실제 결과)
ENTER Answer :  ‘현실에 안주하지 않고 끊임없이 나를 채워가는 것’이는 저를 데이터분석가라는 꿈으로 꾸준히 달려갈 수 있게 해주었습니다. 준비해야할 것들이 많아 막막했지만 단순히 프로그래밍을 잘하는 것이 아닌 ‘데이터에서 최대의 인사이트를 도출하는 프로젝트 해보기’를 세부 목표로 설정하였습니다. 첫째로 KOCW 강의와 프로그래밍 서적으로 R 언어를 독학하였습니다. 통계학 수업에서 배운 내용과 더불어 이론이 프로그래밍 상에서는 어떻게 구현이 되는지에 집중을 하며 언어 환경을 익히려고 노력하였습니다. 이후에는 데이터마이닝 알고리즘이 실제 데이터에 어떻게 활용이 되는지를 중점으로 공부하였습니다. 하지만 그 과정에서 데이터를 이해하고 산업에 맞는 결과를 도출하기보다는 단순 분석만 하는 스스로를 발견하면서 교수님께서 주의하신 분석기계가 되어가고 있다는 생각이 들었습니다. 같은 문제라도 전공과 사람마다 접근하는 방식이 다르기에 다양한 견해를 논의하고 학습하는 장이 있으면 좋겠다는 생각이 들어 동아리를 설립했습니다. 그곳에서 다양한 전공을 가진 학우들과 함께 프로그래밍과 문제를 다각도로 보는 방식을 배울 수 있었습니다. 처음에 데이터분석을 공부하겠다고 결심했을 때 늦지 않았을까 하는 걱정이 들었습니다. 하지만 한 계단 오른 것에 안주하지 않고 다음 단계를 준비하는 자세로 평소에 어려워했던 마케팅 데이터로 기획까지 제안하는 결과물을 완성할 수 있었습니다.
ENTER Question :  n




3638 : 2*귀하의학창시절을 스스로 평가한다면 본인은 어떠한 사람이라고 생각하는지 약술하십시오. 현재 596 Bytes / 600 Bytes / 300자 이내 [다양한 영역에 도전하는] 이공계 통계학도이지만 다양한 분야에 관심을 갖고 도전해왔습니다. 중고등학교시절에는 만화, 심리학, 정신과학 동아리를 결성했습니다. 대학에서는 사회과학을 복수전공했습니다. 또한, 미국에 교환학생을 가서도 미국, 이탈리아, 일본 등 다양한 나라에서 온 친구들과 함께 미술, 국제 경제학 등 다양한 수업을 수강했습니다. 이 과정 속에서 다양한 전공, 그리고다양한 국적의 사람들과 함께했습니다. 이런 학창시절을 통해 새로운 분야, 다양한 사람 속에 무리 없이 녹아드는열린 마음과, 세상을 다양한 시각에서 바라볼 수 있는 넓은 시야를 가진 학생으로 자랐습니다. 3*귀하가생각하는 최고의 고객서비스란 무엇이며, 그 이유에 대해 약술하십시오현재 598 Bytes / 600 Bytes / 300자 이내 [최선의 맞춤 상품을 제시하기] 사람들은 누구나 자신의 금전적 상황 속에서 가장 이득이 되는 선택을 하고 싶어 하지만, 정보가 부족하기에 그러지 못하는 경우가 많습니다. 하지만 개인과달리 KB국민은행은 이미 막대한 양의 금융 데이터를 갖고 있습니다. 그러므로쌓아온 빅데이터를 분석한다면 어떤 성질을 가진 고객은 어떤 선택을 해야 최선의 결과를 만들 수 있는지에 대한 신뢰도 높은 메뉴얼을 만들 수 있을것입니다. 이렇듯 고객 한 분 한 분의 상황에 맞춰 최선의 이익을 만들어 낼 수 있는 금융 상품을 만들어 추천하는 것이, 은행이 은행이 가진 고유의 정보를 이용해 고객을 감동시킬 수 있는 최고의 서비스라고 생각합니다.


ENTER Question :  귀하의 학창시절을 스스로 평가한다면 본인은 어떠한 사람이라고 생각하는지 약술하십시오.
ENTER Answer :  [다양한 영역에 도전하는] 이공계 통계학도이지만 다양한 분야에 관심을 갖고 도전해왔습니다. 중고등학교시절에는 만화, 심리학, 정신과학 동아리를 결성했습니다. 대학에서는 사회과학을 복수전공했습니다. 또한, 미국에 교환학생을 가서도 미국, 이탈리아, 일본 등 다양한 나라에서 온 친구들과 함께 미술, 국제 경제학 등 다양한 수업을 수강했습니다. 이 과정 속에서 다양한 전공, 그리고다양한 국적의 사람들과 함께했습니다. 이런 학창시절을 통해 새로운 분야, 다양한 사람 속에 무리 없이 녹아드는 열린 마음과, 세상을 다양한 시각에서 바라볼 수 있는 넓은 시야를 가진 학생으로 자랐습니다.
ENTER Question :  귀하가 생각하는 최고의 고객 서비스란 무엇이며, 그 이유에 대해 약술하십시오.
ENTER Answer :  [최선의 맞춤 상품을 제시하기] 사람들은 누구나 자신의 금전적 상황 속에서 가장 이득이 되는 선택을 하고 싶어 하지만, 정보가 부족하기에 그러지 못하는 경우가 많습니다. 하지만 개인과달리 KB국민은행은 이미 막대한 양의 금융 데이터를 갖고 있습니다. 그러므로쌓아온 빅데이터를 분석한다면 어떤 성질을 가진 고객은 어떤 선택을 해야 최선의 결과를 만들 수 있는지에 대한 신뢰도 높은 메뉴얼을 만들 수 있을것입니다. 이렇듯 고객 한 분 한 분의 상황에 맞춰 최선의 이익을 만들어 낼 수 있는 금융 상품을 만들어 추천하는 것이, 은행이 은행이 가진 고유의 정보를 이용해 고객을 감동시킬 수 있는 최고의 서비스라고 생각합니다.
ENTER Question :  n




3645 : 2*귀하의학창시절을 스스로 평가한다면 본인은 어떠한 사람이라고 생각하는지 약술하십시오. 현재 596 Bytes / 600 Bytes / 300자 이내 [다양한 영역에 도전하는] 이공계 통계학도이지만 다양한 분야에 관심을 갖고 도전해왔습니다. 중고등학교시절에는 만화, 심리학, 정신과학 동아리를 결성했습니다. 대학에서는 사회과학을 복수전공했습니다. 또한, 미국에 교환학생을 가서도 미국, 이탈리아, 일본 등 다양한 나라에서 온 친구들과 함께 미술, 국제 경제학 등 다양한 수업을 수강했습니다. 이 과정 속에서 다양한 전공, 그리고다양한 국적의 사람들과 함께했습니다. 이런 학창시절을 통해 새로운 분야, 다양한 사람 속에 무리 없이 녹아드는열린 마음과, 세상을 다양한 시각에서 바라볼 수 있는 넓은 시야를 가진 학생으로 자랐습니다. 3*귀하가생각하는 최고의 고객서비스란 무엇이며, 그 이유에 대해 약술하십시오현재 598 Bytes / 600 Bytes / 300자 이내 [최선의 맞춤 상품을 제시하기] 사람들은 누구나 자신의 금전적 상황 속에서 가장 이득이 되는 선택을 하고 싶어 하지만, 정보가 부족하기에 그러지 못하는 경우가 많습니다. 하지만 개인과달리 KB국민은행은 이미 막대한 양의 금융 데이터를 갖고 있습니다. 그러므로쌓아온 빅데이터를 분석한다면 어떤 성질을 가진 고객은 어떤 선택을 해야 최선의 결과를 만들 수 있는지에 대한 신뢰도 높은 메뉴얼을 만들 수 있을것입니다. 이렇듯 고객 한 분 한 분의 상황에 맞춰 최선의 이익을 만들어 낼 수 있는 금융 상품을 만들어 추천하는 것이, 은행이 은행이 가진 고유의 정보를 이용해 고객을 감동시킬 수 있는 최고의 서비스라고 생각합니다.


ENTER Question :  귀하의 학창시절을 스스로 평가한다면 본인은 어떠한 사람이라고 생각하는지 약술하십시오.
ENTER Answer :  [다양한 영역에 도전하는] 이공계 통계학도이지만 다양한 분야에 관심을 갖고 도전해왔습니다. 중고등학교시절에는 만화, 심리학, 정신과학 동아리를 결성했습니다. 대학에서는 사회과학을 복수전공했습니다. 또한, 미국에 교환학생을 가서도 미국, 이탈리아, 일본 등 다양한 나라에서 온 친구들과 함께 미술, 국제 경제학 등 다양한 수업을 수강했습니다. 이 과정 속에서 다양한 전공, 그리고다양한 국적의 사람들과 함께했습니다. 이런 학창시절을 통해 새로운 분야, 다양한 사람 속에 무리 없이 녹아드는 열린 마음과, 세상을 다양한 시각에서 바라볼 수 있는 넓은 시야를 가진 학생으로 자랐습니다.
ENTER Question :  v
ENTER Answer :  [최선의 맞춤 상품을 제시하기] 사람들은 누구나 자신의 금전적 상황 속에서 가장 이득이 되는 선택을 하고 싶어 하지만, 정보가 부족하기에 그러지 못하는 경우가 많습니다. 하지만 개인과달리 KB국민은행은 이미 막대한 양의 금융 데이터를 갖고 있습니다. 그러므로쌓아온 빅데이터를 분석한다면 어떤 성질을 가진 고객은 어떤 선택을 해야 최선의 결과를 만들 수 있는지에 대한 신뢰도 높은 메뉴얼을 만들 수 있을것입니다. 이렇듯 고객 한 분 한 분의 상황에 맞춰 최선의 이익을 만들어 낼 수 있는 금융 상품을 만들어 추천하는 것이, 은행이 은행이 가진 고유의 정보를 이용해 고객을 감동시킬 수 있는 최고의 서비스라고 생각합니다.
ENTER Question :  n




5511 : 3, 최근 사회이슈 중 중요하다고 생각되는 한가지를 선택하고 이에 관한 자신의 견해를 기술해 주시기 바랍니다. 1000 [차세대 자동차산업, 커넥티드 카] 최근 관심 있게 구독한 뉴스는 ‘MCW 에서 BMW - SKT 커넥티드 카(T5)’입니다. 무인자동차를 넘어 세상은 자동차와 인프라, 자동차와 스마트폰을 연결하는 커넥티드 카의 시대로 빠르게 변화하고 있습니다. T5는 ‘커넥티드 카’에 5G 통신 기술이 결합된 차입니다. 이 두 회사 뿐 아니라, LG VC 사업부와 폭스바겐의 공동 플렛폼 개발, 구글을 포함한 세계 적인 기업들이 커넥티드 카의 발전을 주목하고 있었기에 주의 깊게 기사를 읽고 커넥티드 카의 중요한 점을 생각해보았습니다. 커넥티드 카의 첫 번째 중요한 점은 ‘통신시스템의 효율성’이라고 생각합니다. 커넥티드 카는 교통정보를 포함한 다양한 정보를 주고받게 됩니다. 다양한 정보를 통해 운전자의 안전과 편의를 제공함에 있어 통신 시스템의 효율성은 꼭 필요합니다. 수많은 정보를 빠르고 정확하게 주고받을 수 있는 효율적인 통신 시스템을 통해 고객에게 혁신적인 가치를 제공하는 것, 그것이 자동차 산업의 방향이라고 생각합니다. 커넥티드 카의 두 번째 핵심적인 조건은 ‘이미지 센서’ 라고 생각합니다. 커넥티드 카는 환경에 대한 인식을 필요로 합니다. 차량 주위 환경은 사용자의 안전 과 직결됩니다. 그렇기에 환경을 정확하게 인식하는 것은 꼭 필요합니다. 만약 다음과 같은 경우가 발생한다면, 1, 앞서가던 차가 찍은 화면이 뒤따라오던 차에 실시간으로 전송 되는데 간섭으로 인해 화질이 악화 되는 경우. 2, 차량이 알아서 장애물을 피해 주행해야 하는데 장애물을 인식을 못하는 경우. 인명피해로 이어 질 수 있기 때문에 이미지 센서의 발전은 자동차 산업에 꼭 필요하다고 생각합니다. 모바일 기기(갤럭시 시리즈)개발을 통해 세계 모바일 산업을 이끌고 있는 삼성 전자는 또한 이미지 센서의 부품(1.0㎛ 1600만 화소 모바일 CMOS)부문 에서도 선도를 하고 

ENTER Question :  최근 사회이슈 중 중요하다고 생각되는 한가지를 선택하고 이에 관한 자신의 견해를 기술해 주시기 바랍니다.
ENTER Answer :  [차세대 자동차산업, 커넥티드 카] 최근 관심 있게 구독한 뉴스는 ‘MCW 에서 BMW - SKT 커넥티드 카(T5)’입니다. 무인자동차를 넘어 세상은 자동차와 인프라, 자동차와 스마트폰을 연결하는 커넥티드 카의 시대로 빠르게 변화하고 있습니다. T5는 ‘커넥티드 카’에 5G 통신 기술이 결합된 차입니다. 이 두 회사 뿐 아니라, LG VC 사업부와 폭스바겐의 공동 플렛폼 개발, 구글을 포함한 세계 적인 기업들이 커넥티드 카의 발전을 주목하고 있었기에 주의 깊게 기사를 읽고 커넥티드 카의 중요한 점을 생각해보았습니다. 커넥티드 카의 첫 번째 중요한 점은 ‘통신시스템의 효율성’이라고 생각합니다. 커넥티드 카는 교통정보를 포함한 다양한 정보를 주고받게 됩니다. 다양한 정보를 통해 운전자의 안전과 편의를 제공함에 있어 통신 시스템의 효율성은 꼭 필요합니다. 수많은 정보를 빠르고 정확하게 주고받을 수 있는 효율적인 통신 시스템을 통해 고객에게 혁신적인 가치를 제공하는 것, 그것이 자동차 산업의 방향이라고 생각합니다. 커넥티드 카의 두 번째 핵심적인 조건은 ‘이미지 센서’ 라고 생각합니다. 커넥티드 카는 환경에 대한 인식을 필요로 합니다. 차량 주위 환경은 사용자의 안전 과 직결됩니다. 그렇기에 환경을 정확하게 인식하는 것은 꼭 필요합니다. 만약 다음과 같은 경우가 발생한다면, 1, 앞서가던 차가 찍은 화면이 뒤따라오던 차에 실시간으로 전송 되는데 간섭으로 인해 화질이 악화 되는 경우. 2, 차량이 알아서 장애물을 피해 주행해야 하는데 장애물을 인식을 못하는 경우. 인명피해로 이어 질 수 있기 때문에 이미지 센서의 발전은 자동차 산업에 꼭 필요하다고 생각합니다. 모바일 기기(갤럭시 시리즈)개발을 통해 세계 모바일 산업을 이끌고 있는 삼성 전자는 또한 이미지 센서의 부품(1.0㎛ 1600만 화소 모바일 CM



6637 : 1번. 여러분은왜 “기아인”이 되고 싶으신가요? (다음 사항을 포함해서 작성해 주세요.) ①”기아” 또는 “자동차” 관련 특별한 경험이나 인연 ② “기아인”이 본인에게 주는 의미 (1800/2000자) [송도 코리아 스피드 페스티벌]  2014년한 TV 예능 프로그램에서 카레이싱에 대해 방영한 적이 있습니다.TV 화면으로만 봐도 숨막히고 박진감 넘치는 카레이싱에 끌려 저는 당해 카레이싱 대회인KSF(Korea Speed Festival)를 관람하기 위해 인천 송도로 갔습니다. 그곳에서레이싱카 박람회와 운전 및 정비를 직접 체험해 볼 수 있는 각종 프로그램 등에 참석하고 카레이싱을 바로 눈 앞에서 즐길 수 있었습니다. 자동차의 엄청난 속도를 현장에서 직접 보니 저 또한 레이싱카를 타고 달리는 선수들처럼 매우 흥분됐습니다. 그러나 이러한 흥분도 잠시, 송도 경기장이 도심 속 경기장인 만큼중간중간 크고 작은 접촉사고에서부터 차량이 도저히 움직일 수 없을 만큼의 큰 파손사고까지 많은 사고가 있었습니다.당시 출전한 150대의 차량 중에서 무려 40대이상의 차량이 파손사고를 겪었다는 보도도 있었습니다. 그 사고들을 눈 앞에서 직접 보니 문득 이런 생각이들었습니다. 어떻게 하면 자동차를 더 빠르면서도 더 튼튼하고 안전하게 만들 수 있을까? 그렇기에 자연스럽게 신차가 양산되기 전에 안전 등 여러 부분에서 고장 없는 차량 품질을 구현하는 품질 직무에관심을 갖게 되었습니다. [도전할 수 있는 기회]  도전이란무엇일까요? 아무런 대책도 없이 일단 지르고 보는 것을 도전이라고 하는 사람들도 있습니다. 하지만 저는 도전이란 막연한 것이 아닌 철저한 준비를 바탕으로 하는 것이라고 생각합니다. 이러한 생각을 갖고 행동하는 제가 도전적인 사람이라고 생각하고, 그역량을 마음껏 발휘하여 제 자신의 가치를 인정받을 수 있는 곳에서 일하고 싶습니다.  기아자동차는 2016년 시장조사업체 JDPA의 차량 초기품질지수(IQS)에서 전체 33개의 브랜드 중 1위에 올랐습니다. IQS에

ENTER Question :  여러분은 왜 “기아인”이 되고 싶으신가요? (”기아” 또는 “자동차” 관련 특별한 경험이나 인연 / “기아인”이 본인에게 주는 의미)
ENTER Answer :  [송도 코리아 스피드 페스티벌]  2014년한 TV 예능 프로그램에서 카레이싱에 대해 방영한 적이 있습니다.TV 화면으로만 봐도 숨막히고 박진감 넘치는 카레이싱에 끌려 저는 당해 카레이싱 대회인KSF(Korea Speed Festival)를 관람하기 위해 인천 송도로 갔습니다. 그곳에서레이싱카 박람회와 운전 및 정비를 직접 체험해 볼 수 있는 각종 프로그램 등에 참석하고 카레이싱을 바로 눈 앞에서 즐길 수 있었습니다. 자동차의 엄청난 속도를 현장에서 직접 보니 저 또한 레이싱카를 타고 달리는 선수들처럼 매우 흥분됐습니다. 그러나 이러한 흥분도 잠시, 송도 경기장이 도심 속 경기장인 만큼중간중간 크고 작은 접촉사고에서부터 차량이 도저히 움직일 수 없을 만큼의 큰 파손사고까지 많은 사고가 있었습니다.당시 출전한 150대의 차량 중에서 무려 40대이상의 차량이 파손사고를 겪었다는 보도도 있었습니다. 그 사고들을 눈 앞에서 직접 보니 문득 이런 생각이들었습니다. 어떻게 하면 자동차를 더 빠르면서도 더 튼튼하고 안전하게 만들 수 있을까? 그렇기에 자연스럽게 신차가 양산되기 전에 안전 등 여러 부분에서 고장 없는 차량 품질을 구현하는 품질 직무에관심을 갖게 되었습니다. [도전할 수 있는 기회]  도전이란무엇일까요? 아무런 대책도 없이 일단 지르고 보는 것을 도전이라고 하는 사람들도 있습니다. 하지만 저는 도전이란 막연한 것이 아닌 철저한 준비를 바탕으로 하는 것이라고 생각합니다. 이러한 생각을 갖고 행동하는 제가 도전적인 사람이라고 생각하고, 그역량을 마음껏 발휘하여 제 자신의 가치를 인정받을 수 있는 곳에서 일하고 싶습니다.  기아자동차는 2016년 시장조사업체 JDPA의 차량 초기품질지수(IQS)에서 전체 33개의 브랜드 중 1위에 올랐습니다. IQS에서 전체1위를 달성한 것



6670 : 1번. 귀하가금호아시아나그룹(1지망회사)을 지원하게 된 동기에 대해 서술해주십시오. (499/500자) [꾸준함을 넘어 No.1으로]  금호타이어에게서끊임없이 도전하고 항상 최초를 추구하려는 모습을 보았습니다.  금호타이어는우수한 기술력을 기반으로 꾸준한 실적을 내고 있습니다. 특히, 지난 2010년 출시한 ‘마제스티 솔루스’는현재까지도 국내 경쟁제품보다 5~10% 정도 더 우수한 성능을 보이며 소비자들의 호응을 얻고 있습니다. 마제스티 솔루스는 특수 실리카와의 결합력이 극대화된 다기능성 고무를 적용해 기존 제품의 승차감 및 제동 기능을개선한 타이어로써 최첨단 소음 감소기술(KANT)까지 적용되어 정숙 주행까지 가능하게 했습니다. 또한 지난 2014년에는 금호타이어가 국내 최초로 개발한 자가 봉합기술을 적용한 실란트 타이어로도 출시되었습니다.  이처럼금호타이어는 꾸준한 기술 개발을 통해 글로벌 기업으로의 도약을 준비하고 있습니다. 저 역시 제 자신의성장과 발전을 위해 끊임없이 노력하고 있습니다. 금호타이어와 함께 제 자신과 회사의 성장 두 마리 토끼를모두 잡고 싶습니다. 2번. 입사후 10년 내 회사에서 이루고 싶은 목표는 무엇이며, 그것을추구하는 이유와 이를 달성하기 위한 계획을 서술해 주십시오. (500/500자) [소통하는 연구원]  저는 종합적인 능력을 겸비한 전문가가 되겠습니다. 이를 통해 금호타이어가 타이어시장에서 높은 경쟁력을 갖고 품질과 고객 존중으로 상징되는 Global Tire Brand가 되는데 이바지할 것입니다. 저는연구개발 직무에서 전문가가 되기 위해 두 가지 계획을 세웠습니다.  첫째, 소통하는전문가가 되겠습니다. 연구개발 직무는 부서 내 여러 전문가와 지식 및 정보를 공유해야 하기 때문에 의사소통이중요합니다. 올바른 의사소통을 위해 제 생각은 소신 있게 표현하고 다른 사람들의 생각은 겸손하게 받아들이겠습니다. 이러한 자세를 갖고 올바른 의사소통을 통해 구성원간의 완벽한 조화를 이루겠습니다.  둘째, 여러분야의 지식을 

ENTER Question :  귀하가 금호아시아나그룹(1지망회사)을 지원하게 된 동기에 대해 서술해주십시오.
ENTER Answer :  [꾸준함을 넘어 No.1으로]  금호타이어에게서끊임없이 도전하고 항상 최초를 추구하려는 모습을 보았습니다.  금호타이어는우수한 기술력을 기반으로 꾸준한 실적을 내고 있습니다. 특히, 지난 2010년 출시한 ‘마제스티 솔루스’는현재까지도 국내 경쟁제품보다 5~10% 정도 더 우수한 성능을 보이며 소비자들의 호응을 얻고 있습니다. 마제스티 솔루스는 특수 실리카와의 결합력이 극대화된 다기능성 고무를 적용해 기존 제품의 승차감 및 제동 기능을개선한 타이어로써 최첨단 소음 감소기술(KANT)까지 적용되어 정숙 주행까지 가능하게 했습니다. 또한 지난 2014년에는 금호타이어가 국내 최초로 개발한 자가 봉합기술을 적용한 실란트 타이어로도 출시되었습니다.  이처럼금호타이어는 꾸준한 기술 개발을 통해 글로벌 기업으로의 도약을 준비하고 있습니다. 저 역시 제 자신의성장과 발전을 위해 끊임없이 노력하고 있습니다. 금호타이어와 함께 제 자신과 회사의 성장 두 마리 토끼를모두 잡고 싶습니다.
ENTER Question :  입사 후 10년 내 회사에서 이루고 싶은 목표는 무엇이며, 그것을 추구하는 이유와 이를 달성하기 위한 계획을 서술해 주십시오.
ENTER Answer :  [소통하는 연구원]  저는 종합적인 능력을 겸비한 전문가가 되겠습니다. 이를 통해 금호타이어가 타이어시장에서 높은 경쟁력을 갖고 품질과 고객 존중으로 상징되는 Global Tire Brand가 되는데 이바지할 것입니다. 저는연구개발 직무에서 전문가가 되기 위해 두 가지 계획을 세웠습니다.  첫째, 소통하는전문가가 되겠습니다. 연구개발 직무는 부서 내 여러 전문가와 지식 및 정보를 공유해야 하기 때문에 의사소통이중요합니다. 올바른 의사소통을 위해 제 생각은 소신 있게 표현하고 다른 사람들의 생각은 겸손하게 받아들이겠습니다. 이러한 자세를 갖고 올바른 의사소통을 통해 구성원간의



6834 : 1번. 해당회사및 직무에 지원하는 동기에 대해 기술하시오. (992/1000자)  Wafer산업은 모든 반도체 산업의 기반이 되는 산업입니다. 때문에 반도체가 점점 더 중요해질 미래에는 Wafer의 중요성 또한증가할 것입니다. LG실트론은 Wafer 시장에서 경쟁력을갖기 위해 혁신적인 변화를 꾀하는 중입니다. 실리콘 Wafer분야에더 많은 투자를 하고 원가 절감 및 거래처 확대 등을 통해 사업을 내실화하고 있습니다. 또한 공정 분석을통해 생산 리드 타임을 줄이고 공정의 자동화 수준을 높이는 미래형 생산 시스템을 구축하여 생산성을 올리고 있습니다. 이러한 과감한 도전이야 말고 LG실트론이 미래 Wafer 시장에서 국내뿐만 아니라 해외에서도 우수한 경쟁력을 갖는데 필요한 모습이라고 생각합니다.  LG실트론이생산성 극대화에 초점을 맞추고 있는 만큼 단위 공정의 최적화 및 공정의 제약사항 등을 해결하는 업무를 수행하는 공정기술의역할이 매우 중요합니다. LG실트론의 공정기술은 Wafer 생산의모든 것을 총괄하는 역할을 하기 때문에 상당한 전문 지식과 동료들을 격려하여 서로 조화를 이루어 나갈 수 있는 팀워크가 필요합니다.  저는 반도체 및 디스플레이 공정을수강하여 Ingot 제조부터 Wafer 패키징까지의 Wafer제조의 전반적인 과정과 원리에 대해 이해하였습니다. 또한고체화학공학, 나노신소재 등을 수강하여 Si와 GaAs 등 무기재료의 구조와 성질에 대해 잘 알고 있습니다.  저는팀의 앞에 서서 남들 눈에 많이 띄는 역할을 하는 카리스마 리더는 아닙니다. 하지만 팀의 뒤에서 팀원들을밀어주고 받쳐주는 서번트 리더로서, 서로 간 신뢰를 형성시켜 팀원들의 성장 및 발전을 돕고, 궁극적으로 조직성과를 달성하게 합니다. 팀에 대한 애정과 이에 따른열정이 저에게 이러한 리더십을 끌어올려 목표를 뛰어넘을 수 있는 힘을 줍니다.  이러한저의 역량을 통해 Wafer 생산 공정의 효율을 향상시키고 Wafer의수율 또한 증가시켜 LG실트론이 모든 부문에서 전문성과 경제성

ENTER Question :  해당 회사 및 직무에 지원하는 동기에 대해 기술하시오.
ENTER Answer :  Wafer산업은 모든 반도체 산업의 기반이 되는 산업입니다. 때문에 반도체가 점점 더 중요해질 미래에는 Wafer의 중요성 또한증가할 것입니다. LG실트론은 Wafer 시장에서 경쟁력을갖기 위해 혁신적인 변화를 꾀하는 중입니다. 실리콘 Wafer분야에더 많은 투자를 하고 원가 절감 및 거래처 확대 등을 통해 사업을 내실화하고 있습니다. 또한 공정 분석을통해 생산 리드 타임을 줄이고 공정의 자동화 수준을 높이는 미래형 생산 시스템을 구축하여 생산성을 올리고 있습니다. 이러한 과감한 도전이야 말고 LG실트론이 미래 Wafer 시장에서 국내뿐만 아니라 해외에서도 우수한 경쟁력을 갖는데 필요한 모습이라고 생각합니다.  LG실트론이생산성 극대화에 초점을 맞추고 있는 만큼 단위 공정의 최적화 및 공정의 제약사항 등을 해결하는 업무를 수행하는 공정기술의역할이 매우 중요합니다. LG실트론의 공정기술은 Wafer 생산의모든 것을 총괄하는 역할을 하기 때문에 상당한 전문 지식과 동료들을 격려하여 서로 조화를 이루어 나갈 수 있는 팀워크가 필요합니다.  저는 반도체 및 디스플레이 공정을수강하여 Ingot 제조부터 Wafer 패키징까지의 Wafer제조의 전반적인 과정과 원리에 대해 이해하였습니다. 또한고체화학공학, 나노신소재 등을 수강하여 Si와 GaAs 등 무기재료의 구조와 성질에 대해 잘 알고 있습니다.  저는팀의 앞에 서서 남들 눈에 많이 띄는 역할을 하는 카리스마 리더는 아닙니다. 하지만 팀의 뒤에서 팀원들을밀어주고 받쳐주는 서번트 리더로서, 서로 간 신뢰를 형성시켜 팀원들의 성장 및 발전을 돕고, 궁극적으로 조직성과를 달성하게 합니다. 팀에 대한 애정과 이에 따른열정이 저에게 이러한 리더십을 끌어올려 목표를 뛰어넘을 수 있는 힘을 줍니다.  이러한저의 역량을 통해 Wafer 생산 공정의 효율을 향상시키고 Wafer의수율 또한 증가시켜 LG실트론이 모든 부문에



7042 : 1귀하가 회사를 선택하는 기준은 무엇이며 왜 그 기준에 ETUDE가 적합한지 기술하시오. (필수입력사항 최대 800 최소 100) [항상 시도하고, 재미의 가치를 이해하는 곳]  경험은 꿈을 찾기 위해 직접 부딛쳐 보는 과정이라고 생각합니다. 실패를 경험하고도 ‘재미’를 느끼는 일, 즉 저에게 가장 어울리는 일을 찾아가는 소중한 시간이기 때문입니다. 화장품이 궁금했을 때는 화장품 매장에서 판매 일을 했고, 마케팅에 대한 실무적 감각을 익히고 싶어 마케팅대행사에서 6개월간 인턴을 했습니다. 무엇보다 항상 좋은 인상, 배려하는 자세로 주변 사람을 내 사람으로 만들었습니다. 다양한 경험은 무엇이 저랑 어울리는지에 대한 고민으로 이어졌고, ‘재미’를 통해 고객과 소통하는 곳이 이상적인 기업이라는 결론을 도출했습니다. [Fun한 마케팅이 실현되는 곳]  스킨푸드나 미샤 등 경쟁사가 아닌 ETUDE의 영업전략 직무를 지원한 가장 큰 이유는 ‘재미’의 가치를 전략으로 가장 잘 이용하기 때문입니다. 미샤에서의 판매 아르바이트를 통해 화장품 산업의 트렌드를 읽는 감각과 고객을 설득하는 능력을 길렀습니다. 하지만 뚜렷한 콘셉트가 없는 미샤에서 재미를 전략으로 구성하기에는 한계가 있었습니다. 반면 ETUDE의 경우 ‘화장놀이문화 Makeup Play’라는 뚜렷한 콘셉트를 매장에 전략적으로 반영하여 10-20대 여성 시장을 선점했습니다. ‘놀이’라는 전략은 스마트한 소비를 지향하는 고객이 증가하는 시점에서, 브랜드 정체성을 견고하게 만들었습니다. 또한 고객들이 재미있게 제품을 직접 검증할 수 있는 문화를 조성했다고 생각합니다. ETUDE에서 시도하는 것을 두려워하지 않고, 항상 고객의 입장에서 Fun한 가치를 아이디어로 구상하는 전략가가 되겠습니다. 2현재 귀하의 가장 부족한 역량은 무엇이며, 역량개발을 위해 어떤 활동과 노력을 하고 있는지 기술하시오. (필수입력사항 최대 600 최소 100) [화장품은 더이상 하나의 산업이 아닙니다] 화장품 산업에 대한 전문가적인

ENTER Question :  귀하가 회사를 선택하는 기준은 무엇이며 왜 그 기준에 ETUDE가 적합한지 기술하시오.
ENTER Answer :  [항상 시도하고, 재미의 가치를 이해하는 곳]  경험은 꿈을 찾기 위해 직접 부딛쳐 보는 과정이라고 생각합니다. 실패를 경험하고도 ‘재미’를 느끼는 일, 즉 저에게 가장 어울리는 일을 찾아가는 소중한 시간이기 때문입니다. 화장품이 궁금했을 때는 화장품 매장에서 판매 일을 했고, 마케팅에 대한 실무적 감각을 익히고 싶어 마케팅대행사에서 6개월간 인턴을 했습니다. 무엇보다 항상 좋은 인상, 배려하는 자세로 주변 사람을 내 사람으로 만들었습니다. 다양한 경험은 무엇이 저랑 어울리는지에 대한 고민으로 이어졌고, ‘재미’를 통해 고객과 소통하는 곳이 이상적인 기업이라는 결론을 도출했습니다. [Fun한 마케팅이 실현되는 곳]  스킨푸드나 미샤 등 경쟁사가 아닌 ETUDE의 영업전략 직무를 지원한 가장 큰 이유는 ‘재미’의 가치를 전략으로 가장 잘 이용하기 때문입니다. 미샤에서의 판매 아르바이트를 통해 화장품 산업의 트렌드를 읽는 감각과 고객을 설득하는 능력을 길렀습니다. 하지만 뚜렷한 콘셉트가 없는 미샤에서 재미를 전략으로 구성하기에는 한계가 있었습니다. 반면 ETUDE의 경우 ‘화장놀이문화 Makeup Play’라는 뚜렷한 콘셉트를 매장에 전략적으로 반영하여 10-20대 여성 시장을 선점했습니다. ‘놀이’라는 전략은 스마트한 소비를 지향하는 고객이 증가하는 시점에서, 브랜드 정체성을 견고하게 만들었습니다. 또한 고객들이 재미있게 제품을 직접 검증할 수 있는 문화를 조성했다고 생각합니다. ETUDE에서 시도하는 것을 두려워하지 않고, 항상 고객의 입장에서 Fun한 가치를 아이디어로 구상하는 전략가가 되겠습니다.
ENTER Question :  현재 귀하의 가장 부족한 역량은 무엇이며, 역량개발을 위해 어떤 활동과 노력을 하고 있는지 기술하시오.
ENTER Answer :  [화장품은 더이상 하나의 산업이 아닙니다] 화장품 산



7233 : 1번. 자신의성장과정과 학창시절에 대해 기술하여 주십시오. (647/700자) [애정과 열정으로 뛰어넘은 목표]  저는대학교 때 동아리에서 마술공연을 한 적이 있습니다. 전년도 공연의 성적이 좋지 않았기 때문에 저희는전년도보다 30명 이상의 관객을 모으는 것을 목표로 잡았습니다. 관객들앞에서 직접 마술을 선보이는 공연자가 관객들의 환호를 더 많이 받을 수 있기 때문에 선호되는 역할이었습니다. 하지만무대설치나 조명, 홍보 등도 육체적으로 더 힘들지만 그만큼 중요하다는 것을 알았기 때문에 저는 그 역할을맡아 공연자들이 다른 것에 신경 쓰지 않고 최고의 무대를 위해 마음 편히 준비할 수 있게 하였습니다. 그결과, 목표 달성은 물론 객석의 80%를 채울 수 있었습니다.  저는앞에서 팀을 이끄는 카리스마 리더는 아닙니다. 하지만 팀의 뒤에서 팀원들을 밀어주고 받쳐주는 서번트리더로서, 서로 간 신뢰를 형성시켜 팀원들의 성장 및 발전을 돕고, 궁극적으로조직성과를 달성하게 합니다. 팀에 대한 애정과 이에 따른 열정이 저에게 이러한 리더십을 끌어올려 목표를뛰어넘을 수 있는 힘을 줍니다.  저는팀에 대한 애정을 갖고 저의 임무를 열정적으로 해내어 목표를 뛰어넘는 것에서 기쁨을 느낍니다. 제가도움이 될 수 있는 어느 곳에서든, 이러한 저만의 장점을 살려 사회에 목표를 뛰어넘는 성취감을 주는삶을 살고 싶습니다. 2번. 지원직무 관련 자신이 갖춘 역량에 대하여 구체적으로 기술하여 주십시오. (692/700자) [책임감으로 성공하다]  저는군 복무 중 후임을 검정고시에 합격시켰습니다. 제 맞후임은 사회에서 문제를 일으켜 경찰서를 드나들었던문제아였습니다. 공부와 연관이 없던 그에게 공부를 가르치는 것은 상당한 노력과 시간이 필요했습니다.  저는취침시간을 활용해 후임을 합격시키겠다는 의지를 갖고 매일 잠을 줄여가며 공부를 가르쳤습니다. 그 결과후임은 한번에 검정고시에 합격했습니다. 힘든 날도 많았지만 맡은 임무에 대한 책임감을 갖고 포기하지않아 목표를 달성할 수 있었던 것

ENTER Question :  자신의 성장과정과 학창시절에 대해 기술하여 주십시오.
ENTER Answer :  [애정과 열정으로 뛰어넘은 목표]  저는대학교 때 동아리에서 마술공연을 한 적이 있습니다. 전년도 공연의 성적이 좋지 않았기 때문에 저희는전년도보다 30명 이상의 관객을 모으는 것을 목표로 잡았습니다. 관객들앞에서 직접 마술을 선보이는 공연자가 관객들의 환호를 더 많이 받을 수 있기 때문에 선호되는 역할이었습니다. 하지만무대설치나 조명, 홍보 등도 육체적으로 더 힘들지만 그만큼 중요하다는 것을 알았기 때문에 저는 그 역할을맡아 공연자들이 다른 것에 신경 쓰지 않고 최고의 무대를 위해 마음 편히 준비할 수 있게 하였습니다. 그결과, 목표 달성은 물론 객석의 80%를 채울 수 있었습니다.  저는앞에서 팀을 이끄는 카리스마 리더는 아닙니다. 하지만 팀의 뒤에서 팀원들을 밀어주고 받쳐주는 서번트리더로서, 서로 간 신뢰를 형성시켜 팀원들의 성장 및 발전을 돕고, 궁극적으로조직성과를 달성하게 합니다. 팀에 대한 애정과 이에 따른 열정이 저에게 이러한 리더십을 끌어올려 목표를뛰어넘을 수 있는 힘을 줍니다.  저는팀에 대한 애정을 갖고 저의 임무를 열정적으로 해내어 목표를 뛰어넘는 것에서 기쁨을 느낍니다. 제가도움이 될 수 있는 어느 곳에서든, 이러한 저만의 장점을 살려 사회에 목표를 뛰어넘는 성취감을 주는삶을 살고 싶습니다.
ENTER Question :  지원직무 관련 자신이 갖춘 역량에 대하여 구체적으로 기술하여 주십시오.
ENTER Answer :  [책임감으로 성공하다]  저는군 복무 중 후임을 검정고시에 합격시켰습니다. 제 맞후임은 사회에서 문제를 일으켜 경찰서를 드나들었던문제아였습니다. 공부와 연관이 없던 그에게 공부를 가르치는 것은 상당한 노력과 시간이 필요했습니다.  저는취침시간을 활용해 후임을 합격시키겠다는 의지를 갖고 매일 잠을 줄여가며 공부를 가르쳤습니다. 그 결과후임은 한번에 검정고시에 합격했습니다. 힘든 날도 많았지만 맡은 임무에 대한 



7382 : 1번. 자신에게주어졌던 일 중 가장 어려웠던 경험은 무엇이었습니까? 그 일을 하게 된 이유와 그때 느꼈던 감정, 진행하면서 가장 어려웠던 점과 그것을 극복하기 위해 했던 행동과 생각, 결과에대해 최대한 구체적으로 작성해 주십시오. (932/1000자) [책임감으로 성공하다]  저는군 복무 중 후임을 가르쳐 검정고시에 합격시켰습니다. 처음 중대장님께 이 얘기를 들었을 때는 몹시 억울했습니다. 상급자의 명령이라 거절할 수도 없고 단지 중대 내에서 최고 학벌이라는 이유로 공부를 가르쳐야 했기 때문입니다. 제 맞후임은 사회에서 크고 작은 문제를 일으켜 경찰서를 드나들었던 소위 말하는 문제아였습니다. 공부와는 연관이 없던 그에게 공부를 가르치는 것은 상당한 노력과 시간이 필요했습니다. 하지만 당시 군인으로서 개인 업무가 있었기 때문에 시간을 내는 것은 쉽지 않은 일이었습니다.  이에저는 취침시간을 활용했습니다. 군인들에게 취침시간은 일과로 인해 쌓인 피로를 풀 수 있는 꿀맛 같은시간입니다. 또한 평소 잠이 많았던 저에게 늦은 시간까지 깨어있는 것은 상당히 힘든 일이었습니다. 하지만 제가 맡은 일이고 한번 해보자는 의지를 갖고 매일 2시간씩열심히 공부를 가르쳤습니다. 후임에게 공부를 가르치기 전에 저는 먼저 제 스스로 공부를 하여 핵심내용들을정리했습니다. 공부를 가르치다 보니 점점 이 일이 재미있어졌고 후임을 반드시 합격시켜야겠다는 책임감또한 생겼습니다. 그렇게 열정을 갖고 공부를 가르친 결과 후임은 단 한번에 검정고시에 합격했습니다. 물론 지쳐서 포기하고 싶은 날도 많았지만 그때마다 주어진 임무에 대한 책임감을 갖고 포기하지 않아 목표를 달성할수 있었던 것 같습니다.  이경험은 제 삶에 많은 변화를 가져다 주었습니다. 언제 어디서든 맡은 일에 대한 책임감을 갖게 되었고무슨 일을 하든 남들보다 열정적으로 더 많은 임무를 수행할 수 있는 강인한 체력을 갖게 되었으며 어떤 어려운 일이라도 반드시 해내겠다는 근성을갖게 되었습니다. 저는 이를 바탕으로 그 누구보

ENTER Question :  자신에게 주어졌던 일 중 가장 어려웠던 경험은 무엇이었습니까? 그 일을 하게 된 이유와 그때 느꼈던 감정, 진행하면서 가장 어려웠던 점과 그것을 극복하기 위해 했던 행동과 생각, 결과에 대해 최대한 구체적으로 작성해 주십시오.
ENTER Answer :  [책임감으로 성공하다]  저는군 복무 중 후임을 가르쳐 검정고시에 합격시켰습니다. 처음 중대장님께 이 얘기를 들었을 때는 몹시 억울했습니다. 상급자의 명령이라 거절할 수도 없고 단지 중대 내에서 최고 학벌이라는 이유로 공부를 가르쳐야 했기 때문입니다. 제 맞후임은 사회에서 크고 작은 문제를 일으켜 경찰서를 드나들었던 소위 말하는 문제아였습니다. 공부와는 연관이 없던 그에게 공부를 가르치는 것은 상당한 노력과 시간이 필요했습니다. 하지만 당시 군인으로서 개인 업무가 있었기 때문에 시간을 내는 것은 쉽지 않은 일이었습니다.  이에저는 취침시간을 활용했습니다. 군인들에게 취침시간은 일과로 인해 쌓인 피로를 풀 수 있는 꿀맛 같은시간입니다. 또한 평소 잠이 많았던 저에게 늦은 시간까지 깨어있는 것은 상당히 힘든 일이었습니다. 하지만 제가 맡은 일이고 한번 해보자는 의지를 갖고 매일 2시간씩열심히 공부를 가르쳤습니다. 후임에게 공부를 가르치기 전에 저는 먼저 제 스스로 공부를 하여 핵심내용들을정리했습니다. 공부를 가르치다 보니 점점 이 일이 재미있어졌고 후임을 반드시 합격시켜야겠다는 책임감또한 생겼습니다. 그렇게 열정을 갖고 공부를 가르친 결과 후임은 단 한번에 검정고시에 합격했습니다. 물론 지쳐서 포기하고 싶은 날도 많았지만 그때마다 주어진 임무에 대한 책임감을 갖고 포기하지 않아 목표를 달성할수 있었던 것 같습니다.  이 경험은 제 삶에 많은 변화를 가져다 주었습니다. 언제 어디서든 맡은 일에 대한 책임감을 갖게 되었고 무슨 일을 하든 남들보다 열정적으로 더 많은 임무를 수행할 수 있는 강인한 체력을 갖게 되었으며 어떤 어려운 일이라도 반드시 해내겠다는 근성을 갖게 되었습니다. 저는



7470 : 1번. 자신에게주어졌던 일 중 가장 어려웠던 경험은 무엇이었습니까? 그 일을 하게 된 이유와 그때 느꼈던 감정, 진행하면서 가장 어려웠던 점과 그것을 극복하기 위해 했던 행동과 생각, 결과에대해 최대한 구체적으로 작성해 주십시오. (932/1000자) [책임감으로 성공하다]  저는군 복무 중 후임을 가르쳐 검정고시에 합격시켰습니다. 처음 중대장님께 이 얘기를 들었을 때는 몹시 억울했습니다. 상급자의 명령이라 거절할 수도 없고 단지 중대 내에서 최고 학벌이라는 이유로 공부를 가르쳐야 했기 때문입니다. 제 맞후임은 사회에서 크고 작은 문제를 일으켜 경찰서를 드나들었던 소위 말하는 문제아였습니다. 공부와는 연관이 없던 그에게 공부를 가르치는 것은 상당한 노력과 시간이 필요했습니다. 하지만 당시 군인으로서 개인 업무가 있었기 때문에 시간을 내는 것은 쉽지 않은 일이었습니다.  이에저는 취침시간을 활용했습니다. 군인들에게 취침시간은 일과로 인해 쌓인 피로를 풀 수 있는 꿀맛 같은시간입니다. 또한 평소 잠이 많았던 저에게 늦은 시간까지 깨어있는 것은 상당히 힘든 일이었습니다. 하지만 제가 맡은 일이고 한번 해보자는 의지를 갖고 매일 2시간씩열심히 공부를 가르쳤습니다. 후임에게 공부를 가르치기 전에 저는 먼저 제 스스로 공부를 하여 핵심내용들을정리했습니다. 공부를 가르치다 보니 점점 이 일이 재미있어졌고 후임을 반드시 합격시켜야겠다는 책임감또한 생겼습니다. 그렇게 열정을 갖고 공부를 가르친 결과 후임은 단 한번에 검정고시에 합격했습니다. 물론 지쳐서 포기하고 싶은 날도 많았지만 그때마다 주어진 임무에 대한 책임감을 갖고 포기하지 않아 목표를 달성할수 있었던 것 같습니다.  이경험은 제 삶에 많은 변화를 가져다 주었습니다. 언제 어디서든 맡은 일에 대한 책임감을 갖게 되었고무슨 일을 하든 남들보다 열정적으로 더 많은 임무를 수행할 수 있는 강인한 체력을 갖게 되었으며 어떤 어려운 일이라도 반드시 해내겠다는 근성을갖게 되었습니다. 저는 이를 바탕으로 그 누구보

ENTER Question :  자신에게 주어졌던 일 중 가장 어려웠던 경험은 무엇이었습니까? 그 일을 하게 된 이유와 그때 느꼈던 감정, 진행하면서 가장 어려웠던 점과 그것을 극복하기 위해 했던 행동과 생각, 결과에 대해 최대한 구체적으로 작성해 주십시오.
ENTER Answer :  [책임감으로 성공하다]  저는군 복무 중 후임을 가르쳐 검정고시에 합격시켰습니다. 처음 중대장님께 이 얘기를 들었을 때는 몹시 억울했습니다. 상급자의 명령이라 거절할 수도 없고 단지 중대 내에서 최고 학벌이라는 이유로 공부를 가르쳐야 했기 때문입니다. 제 맞후임은 사회에서 크고 작은 문제를 일으켜 경찰서를 드나들었던 소위 말하는 문제아였습니다. 공부와는 연관이 없던 그에게 공부를 가르치는 것은 상당한 노력과 시간이 필요했습니다. 하지만 당시 군인으로서 개인 업무가 있었기 때문에 시간을 내는 것은 쉽지 않은 일이었습니다.  이에저는 취침시간을 활용했습니다. 군인들에게 취침시간은 일과로 인해 쌓인 피로를 풀 수 있는 꿀맛 같은시간입니다. 또한 평소 잠이 많았던 저에게 늦은 시간까지 깨어있는 것은 상당히 힘든 일이었습니다. 하지만 제가 맡은 일이고 한번 해보자는 의지를 갖고 매일 2시간씩열심히 공부를 가르쳤습니다. 후임에게 공부를 가르치기 전에 저는 먼저 제 스스로 공부를 하여 핵심내용들을정리했습니다. 공부를 가르치다 보니 점점 이 일이 재미있어졌고 후임을 반드시 합격시켜야겠다는 책임감또한 생겼습니다. 그렇게 열정을 갖고 공부를 가르친 결과 후임은 단 한번에 검정고시에 합격했습니다. 물론 지쳐서 포기하고 싶은 날도 많았지만 그때마다 주어진 임무에 대한 책임감을 갖고 포기하지 않아 목표를 달성할수 있었던 것 같습니다.  이 경험은 제 삶에 많은 변화를 가져다 주었습니다. 언제 어디서든 맡은 일에 대한 책임감을 갖게 되었고 무슨 일을 하든 남들보다 열정적으로 더 많은 임무를 수행할 수 있는 강인한 체력을 갖게 되었으며 어떤 어려운 일이라도 반드시 해내겠다는 근성을 갖게 되었습니다. 저는



7730 : 1번 문항 [제목은 개인적 내용으로 삭제하였습니다] 물류업계에서 확실한 우위로 직원과 함께 성장할 수 있는 회사를 찾았습니다. 그 곳의 중심에는 대한통운이 있었습니다. 대한통운은 시대의 흐름에 발맞추고 미래에 대한 철저한 대비로 물류업계를 선도하고 있습니다. 또한, 다른 기업이 생각해 내지 못했던 항로 운항을 통하여 기존항로보다 시간과 비용을 절약하며 가격경쟁력을 높인 획기적인 아이디어로 크게 성장하였습니다. 이는 제 전공 산업공학이 추구하는 목표 방향과 같다는 확신을 하고 지원하게 되었습니다 산업공학도로서 종합적인 시스템을 파악할 수 있는 인재로 성장하였습니다. 그리고 통계학적 분석과 결합하여서 리드 타임 축소, 원가 절감, 등 물류사업에서 필요할 효과적일 기법들을 프로젝트를 통해 경험해보았습니다. 이는 대한통운 물류운영부서에서 추구하는 방향과 같다고 생각합니다. 같은 목적을 추구하였던 만큼 물류운영직무에서 꼭 필요로 하는 인재가 될 수 있다고 믿어 지원하였습니다. 2번 문항 [] 대한통운 물동량 증가를 이어가게끔 하고 3자 물류를 확대하고 있는 흐름에 발 빠르게 맞춰나가겠습니다. 특히 물류 산업의 경우 국내 물류 산업이 포화로 인해 해외 거점 확대에 집중하고 있습니다. 이런 경향까지 모두 받아들여 적절하게 대처하고 확실한 통합물류 서비스를 제공할 수 있는 능력을 지닌 인재가 되겠습니다. 특히ㅇㅇ 수송 물량의 비중을 증가시켜 신규 해외 고객 확보를 통해 해외 네트워크를 구축하여 이익 극대화에 힘쓰겠습니다. 또한, 고객의 요구사항을 모두 받아들여 고객이 만족할 수 있는 통합물류 서비스를 제공으로 대한통운을 성장시키겠습니다. [평범하지 않겠습니다] 이를 이루기 위해 입사하자마자 물류 프로세스에 대해 철저히 익히겠습니다. 기본이 바로 잡혀있는 것을 바탕으로 해야 자기발전이 가능하기 때문입니다. 물류프로세스에 대한 완벽한 이해로 합리적인 계획을 수립하고, 첨단화된 장비들도 빠르게 익혀서 업무에 빠르게 적응하겠습니다. 그 후 대리를 달 때까지 전문 용

ENTER Question :  지원 동기 및 입사 후 포부
ENTER Answer :  물류업계에서 확실한 우위로 직원과 함께 성장할 수 있는 회사를 찾았습니다. 그 곳의 중심에는 대한통운이 있었습니다. 대한통운은 시대의 흐름에 발맞추고 미래에 대한 철저한 대비로 물류업계를 선도하고 있습니다. 또한, 다른 기업이 생각해 내지 못했던 항로 운항을 통하여 기존항로보다 시간과 비용을 절약하며 가격경쟁력을 높인 획기적인 아이디어로 크게 성장하였습니다. 이는 제 전공 산업공학이 추구하는 목표 방향과 같다는 확신을 하고 지원하게 되었습니다 산업공학도로서 종합적인 시스템을 파악할 수 있는 인재로 성장하였습니다. 그리고 통계학적 분석과 결합하여서 리드 타임 축소, 원가 절감, 등 물류사업에서 필요할 효과적일 기법들을 프로젝트를 통해 경험해보았습니다. 이는 대한통운 물류운영부서에서 추구하는 방향과 같다고 생각합니다. 같은 목적을 추구하였던 만큼 물류운영직무에서 꼭 필요로 하는 인재가 될 수 있다고 믿어 지원하였습니다. 2번 문항 [] 대한통운 물동량 증가를 이어가게끔 하고 3자 물류를 확대하고 있는 흐름에 발 빠르게 맞춰나가겠습니다. 특히 물류 산업의 경우 국내 물류 산업이 포화로 인해 해외 거점 확대에 집중하고 있습니다. 이런 경향까지 모두 받아들여 적절하게 대처하고 확실한 통합물류 서비스를 제공할 수 있는 능력을 지닌 인재가 되겠습니다. 특히ㅇㅇ 수송 물량의 비중을 증가시켜 신규 해외 고객 확보를 통해 해외 네트워크를 구축하여 이익 극대화에 힘쓰겠습니다. 또한, 고객의 요구사항을 모두 받아들여 고객이 만족할 수 있는 통합물류 서비스를 제공으로 대한통운을 성장시키겠습니다. [평범하지 않겠습니다] 이를 이루기 위해 입사하자마자 물류 프로세스에 대해 철저히 익히겠습니다. 기본이 바로 잡혀있는 것을 바탕으로 해야 자기발전이 가능하기 때문입니다. 물류프로세스에 대한 완벽한 이해로 합리적인 계획을 수립하고, 첨단화된 장비들도 빠르게 익혀서 업무에 빠르게 적응하겠습니다. 그 후 대리



7978 : 1지원동기 : 지원동기를 구체적으로 기술해주세요500 <너와 나의 연결고리> 자랑스러운 업계 1위, 롯데칠성음료가 가진 브랜드의 가치는 상상이상입니다. 롯데주류는 항상 시대마다 달랐던 시장의 트렌드를 먼저 읽고 이끄는 선구적인 역할을 해왔습니다. 특히, 2015년에 출시된 ‘순하리’는 소위 말하는 ‘대박’이었습니다. 이렇듯, 트렌드를 앞서 나가는 롯데의 안목에는 말하지 않아도 끊임없는 연구와 노력이란 기반이 있었을 것이라 사료됩니다. 이처럼, 끊임없이 기업의 목표를 실현하려는 노력의 정신은 제 삶에 있어 가장 중요한 ‘노력을 통한 목표 달성’의 가치에 상응하다고 생각합니다. 높은 학점과 어학 점수를 통한 장학금 취득, 교환학생 선발 등이 증명하듯 저는 노력이라는 전략을 이용하여 원하는 결과를 수립합니다. 이는, ‘열정’이라는 가치를 중요시 여기는 롯데가 걸어가는 길과 제가 한 평생 걸어왔고, 앞으로 걸어갈 저의 길과 일맥상통하다고 생각합니다. 같은 가치를 가진 롯데란 울타리에서 꿈을 펼치고 싶습니다. 2성장과정 : 성장과정을 구체적으로 기술해주세요.800 <자기개발의 즐거움을 아는 자로서의 성장> 저는 ‘자신의 일은 스스로 하자’라는 부모님의 교육 아래, 독립심 있고, 책임감이 강한 사람으로 성장하였습니다. 또한, 공부하면서 얻는 앎의 쾌락을 즐기며 자기개발의 재미를 느끼게 되었습니다. 예로서, 이러한 공부에 대한 재미와 승부욕은 항상 2등만 하던 저를 졸업 전 가장 중요한 마지막 시험에서 비로소 1등으로 만들어줘, 비평준화 지역에서 공립 중 가장 높은 고등학교에 진학할 수 있었습니다. 또한, 대학교 진학 후엔 교환학생이라는 꿈을 가지게 되었고 그 꿈을 이루기 위해서 제가 평소에 잘 하지 못하는 영어라는 문턱을 넘어보기로 했습니다. 학원을 오가는 지하철, 그리고 아르바이트 쉬는 시간과 같은 낭비하기 쉬운 시간까지 이용하여 공부하였고, 잠을 줄이면서 영어공부에 매진하였습니다. 그렇게 꾸준히 6개월을 공부한 결과, 원하는 점수를 얻어 교환학생이 될

ENTER Question :  지원동기를 구체적으로 기술해주세요
ENTER Answer :  [너와 나의 연결고리] 자랑스러운 업계 1위, 롯데칠성음료가 가진 브랜드의 가치는 상상이상입니다. 롯데주류는 항상 시대마다 달랐던 시장의 트렌드를 먼저 읽고 이끄는 선구적인 역할을 해왔습니다. 특히, 2015년에 출시된 ‘순하리’는 소위 말하는 ‘대박’이었습니다. 이렇듯, 트렌드를 앞서 나가는 롯데의 안목에는 말하지 않아도 끊임없는 연구와 노력이란 기반이 있었을 것이라 사료됩니다. 이처럼, 끊임없이 기업의 목표를 실현하려는 노력의 정신은 제 삶에 있어 가장 중요한 ‘노력을 통한 목표 달성’의 가치에 상응하다고 생각합니다. 높은 학점과 어학 점수를 통한 장학금 취득, 교환학생 선발 등이 증명하듯 저는 노력이라는 전략을 이용하여 원하는 결과를 수립합니다. 이는, ‘열정’이라는 가치를 중요시 여기는 롯데가 걸어가는 길과 제가 한 평생 걸어왔고, 앞으로 걸어갈 저의 길과 일맥상통하다고 생각합니다. 같은 가치를 가진 롯데란 울타리에서 꿈을 펼치고 싶습니다.
ENTER Question :  성장과정을 구체적으로 기술해주세요.
ENTER Answer :  [자기개발의 즐거움을 아는 자로서의 성장] 저는 ‘자신의 일은 스스로 하자’라는 부모님의 교육 아래, 독립심 있고, 책임감이 강한 사람으로 성장하였습니다. 또한, 공부하면서 얻는 앎의 쾌락을 즐기며 자기개발의 재미를 느끼게 되었습니다. 예로서, 이러한 공부에 대한 재미와 승부욕은 항상 2등만 하던 저를 졸업 전 가장 중요한 마지막 시험에서 비로소 1등으로 만들어줘, 비평준화 지역에서 공립 중 가장 높은 고등학교에 진학할 수 있었습니다. 또한, 대학교 진학 후엔 교환학생이라는 꿈을 가지게 되었고 그 꿈을 이루기 위해서 제가 평소에 잘 하지 못하는 영어라는 문턱을 넘어보기로 했습니다. 학원을 오가는 지하철, 그리고 아르바이트 쉬는 시간과 같은 낭비하기 쉬운 시간까지 이용하여 공부하였고, 잠을 줄이면서 영어공부에 매진하였습니다. 그렇

### [3-3] 고친 내용 업데이트

In [103]:
for idx, con in new_contents:
    content2[idx] = con

for idx, con in zip(temp, changes):
    content2[idx] = con

### [3-4] 숫자가 아닌 애들 먼저 출력

In [107]:
df['NumUpdate'] = content2

In [108]:
df

UserInfos  \
0     한양대 ERICA / 기계공학과 / 학점 3.78/4.5 / 토익: 855, 토익스...   
1     지방사립 / 경제학 / 학점 4.15/4.5 / 토익: 720, 토익스피킹: 140...   
2     한서삼 / 컴퓨터공학과 / 학점 4.44 / 토익: 765, 토익스피킹: , 오픽:...   
3     인하대 / 컴퓨터공학과 / 학점 3.36 / 토익: , 토익스피킹: , 오픽: IM...   
4     경기대학교 / 응용통계학과 / 학점 3.8 / 토익: 705, 토익스피킹: , 오픽...   
...                                                 ...   
9874                 부산경상대학교 / 광고마케팅과 / 학점 3.8 / 토익 850   
9875  상명대학교 / 국제통상학 / 학점 4.3 / 토익 900, HSK 6급 / 해외인턴...   
9876                             지방전문대 / 공업화학 / 학점 3.06   
9877      지거국 / 전자공학 / 학점 3.7 / 토익 900대 / 음악봉사활동, 멘토링 등   
9878        지거국 / 전자공학 / 학점 3.7 / 토익 900 / 음악봉사활동, 멘토링등   

                                                Content  \
0     본인이 회사를 선택할때의 기준은 무엇이며, 왜 현대엔지니어링이 그 기준에 적합한지를...   
1     [금융 백종원] 백종원은 여러 매체를 통해 요리에 관한 각종 정보와 비결을 공유하며...   
2     현대엠엔소프트에 지원한 이유가 무엇인가요? (회사 및 직무에 대한 흥미와 커리어 계...   
3     본인이 이수한 SW 관련 전공 중, 가장 즐겁게 배웠던 수업 혹은 가장 좋은 성적을...   
4     1. 해당 지원분야 지원 동기와 해당 사업 참가 후 계획을 기술해주십시오(500자 ...   
...                                                 ...   
9874  1. 지원동기\n\n인재와 기업이 함께 성장할 수 있는 곳, 그곳이 바로 웅진코웨이...   
9875  1. 현대자동차에 지원하게 된 동기 또는 해당 직무에 지원하게 된 동기는 무엇입니까...   
9876  성격소개\n\n대부분의 사람들이 평가하는 첫인상은 소심하고 내성적인 성격일것이라 생...   
9877  [질문 1] 취업이라는 목표 이외에 인생의 큰 목표를 세운 경험, 그 목표를 실현하...   
9878  한전 자가소개서 양식 500자 12줄 이내\n\n[질문 1] 취업이라는 목표 이외에...   

                                                    URL  CompanyName  \
0     https://linkareer.com/cover-letter/25204?page=...      현대엔지니어링   
1     https://linkareer.com/cover-letter/25579?page=...         삼성생명   
2     https://linkareer.com/cover-letter/28660?page=...      현대엠엔소프트   
3     https://linkareer.com/cover-letter/28662?page=...      네이버파이낸셜   
4     https://linkareer.com/cover-letter/28755?page=...  한국지능정보사회지능원   
...                                                 ...          ...   
9874  https://linkareer.com/cover-letter/13306?page=...        웅진코웨이   
9875  https://linkareer.com/cover-letter/20654?page=...        현대자동차   
9876  https://linkareer.com/cover-letter/22028?page=...           심팩   
9877  https://linkareer.com/cover-letter/22089?page=...         한국전력   
9878  https://linkareer.com/cover-letter/17456?page=...         한국전력   

                       CompanyCareer CompanyDate  \
0                          자산관리 자산설비    2021 상반기   
1                             지점사무지원    2021 상반기   
2                       영상처리 및 컴퓨터비전    2021 상반기   
3     [NAVER FINANCIAL] 하계 SW개발자 인턴십    2021 상반기   
4                   데이터 분석 및 운영 청년인턴    2021 상반기   
...                              ...         ...   
9874                             마케팅    2013 하반기   
9875                            영업관리    2013 상반기   
9876                            용접관리    2013 상반기   
9877                        인턴[송,배전]    2013 상반기   
9878                        인턴[송,배전]    2013 상반기   

                                              NumUpdate  
0     본인이 회사를 선택할때의 기준은 무엇이며, 왜 현대엔지니어링이 그 기준에 적합한지를...  
1     [금융 백종원] 백종원은 여러 매체를 통해 요리에 관한 각종 정보와 비결을 공유하며...  
2     현대엠엔소프트에 지원한 이유가 무엇인가요? (회사 및 직무에 대한 흥미와 커리어 계...  
3     본인이 이수한 SW 관련 전공 중, 가장 즐겁게 배웠던 수업 혹은 가장 좋은 성적을...  
4     {'해당 지원분야 지원 동기와 해당 사업 참가 후 계획을 기술해주십시오(500자 내...  
...                                                 ...  
9874  {'지원동기': '인재와 기업이 함께 성장할 수 있는 곳, 그곳이 바로 웅진코웨이라...  
9875  {'현대자동차에 지원하게 된 동기 또는 해당 직무에 지원하게 된 동기는 무엇입니까?...  
9876  성격소개@@@대부분의 사람들이 평가하는 첫인상은 소심하고 내성적인 성격일것이라 생각...  
9877  [질문 1] 취업이라는 목표 이외에 인생의 큰 목표를 세운 경험, 그 목표를 실현하...  
9878  한전 자가소개서 양식 500자 12줄 이내@@@[질문 1] 취업이라는 목표 이외에 ...  

[9879 rows x 7 columns]

In [115]:
start_num_list = [con[0] for con in start_num]
T_F = [True if idx in start_num_list else False for idx in range(len(df))]

In [116]:
start_num_list[:10]

[4, 6, 9, 10, 11, 15, 16, 18, 20, 23]

In [117]:
T_F[:30]

[False,
 False,
 False,
 False,
 True,
 False,
 True,
 False,
 False,
 True,
 True,
 True,
 False,
 False,
 False,
 True,
 True,
 False,
 True,
 False,
 True,
 False,
 False,
 True,
 False,
 False,
 True,
 False,
 True,
 False]

In [118]:
df['NotUpdate'] = T_F

In [119]:
df

UserInfos  \
0     한양대 ERICA / 기계공학과 / 학점 3.78/4.5 / 토익: 855, 토익스...   
1     지방사립 / 경제학 / 학점 4.15/4.5 / 토익: 720, 토익스피킹: 140...   
2     한서삼 / 컴퓨터공학과 / 학점 4.44 / 토익: 765, 토익스피킹: , 오픽:...   
3     인하대 / 컴퓨터공학과 / 학점 3.36 / 토익: , 토익스피킹: , 오픽: IM...   
4     경기대학교 / 응용통계학과 / 학점 3.8 / 토익: 705, 토익스피킹: , 오픽...   
...                                                 ...   
9874                 부산경상대학교 / 광고마케팅과 / 학점 3.8 / 토익 850   
9875  상명대학교 / 국제통상학 / 학점 4.3 / 토익 900, HSK 6급 / 해외인턴...   
9876                             지방전문대 / 공업화학 / 학점 3.06   
9877      지거국 / 전자공학 / 학점 3.7 / 토익 900대 / 음악봉사활동, 멘토링 등   
9878        지거국 / 전자공학 / 학점 3.7 / 토익 900 / 음악봉사활동, 멘토링등   

                                                Content  \
0     본인이 회사를 선택할때의 기준은 무엇이며, 왜 현대엔지니어링이 그 기준에 적합한지를...   
1     [금융 백종원] 백종원은 여러 매체를 통해 요리에 관한 각종 정보와 비결을 공유하며...   
2     현대엠엔소프트에 지원한 이유가 무엇인가요? (회사 및 직무에 대한 흥미와 커리어 계...   
3     본인이 이수한 SW 관련 전공 중, 가장 즐겁게 배웠던 수업 혹은 가장 좋은 성적을...   
4     1. 해당 지원분야 지원 동기와 해당 사업 참가 후 계획을 기술해주십시오(500자 ...   
...                                                 ...   
9874  1. 지원동기\n\n인재와 기업이 함께 성장할 수 있는 곳, 그곳이 바로 웅진코웨이...   
9875  1. 현대자동차에 지원하게 된 동기 또는 해당 직무에 지원하게 된 동기는 무엇입니까...   
9876  성격소개\n\n대부분의 사람들이 평가하는 첫인상은 소심하고 내성적인 성격일것이라 생...   
9877  [질문 1] 취업이라는 목표 이외에 인생의 큰 목표를 세운 경험, 그 목표를 실현하...   
9878  한전 자가소개서 양식 500자 12줄 이내\n\n[질문 1] 취업이라는 목표 이외에...   

                                                    URL  CompanyName  \
0     https://linkareer.com/cover-letter/25204?page=...      현대엔지니어링   
1     https://linkareer.com/cover-letter/25579?page=...         삼성생명   
2     https://linkareer.com/cover-letter/28660?page=...      현대엠엔소프트   
3     https://linkareer.com/cover-letter/28662?page=...      네이버파이낸셜   
4     https://linkareer.com/cover-letter/28755?page=...  한국지능정보사회지능원   
...                                                 ...          ...   
9874  https://linkareer.com/cover-letter/13306?page=...        웅진코웨이   
9875  https://linkareer.com/cover-letter/20654?page=...        현대자동차   
9876  https://linkareer.com/cover-letter/22028?page=...           심팩   
9877  https://linkareer.com/cover-letter/22089?page=...         한국전력   
9878  https://linkareer.com/cover-letter/17456?page=...         한국전력   

                       CompanyCareer CompanyDate  \
0                          자산관리 자산설비    2021 상반기   
1                             지점사무지원    2021 상반기   
2                       영상처리 및 컴퓨터비전    2021 상반기   
3     [NAVER FINANCIAL] 하계 SW개발자 인턴십    2021 상반기   
4                   데이터 분석 및 운영 청년인턴    2021 상반기   
...                              ...         ...   
9874                             마케팅    2013 하반기   
9875                            영업관리    2013 상반기   
9876                            용접관리    2013 상반기   
9877                        인턴[송,배전]    2013 상반기   
9878                        인턴[송,배전]    2013 상반기   

                                              NumUpdate  NotUpdate  
0     본인이 회사를 선택할때의 기준은 무엇이며, 왜 현대엔지니어링이 그 기준에 적합한지를...      False  
1     [금융 백종원] 백종원은 여러 매체를 통해 요리에 관한 각종 정보와 비결을 공유하며...      False  
2     현대엠엔소프트에 지원한 이유가 무엇인가요? (회사 및 직무에 대한 흥미와 커리어 계...      False  
3     본인이 이수한 SW 관련 전공 중, 가장 즐겁게 배웠던 수업 혹은 가장 좋은 성적을...      False  
4     {'해당 지원분야 지원 동기와 해당 사업 참가 후 계획을 기술해주십시오(500자 내...       True  
...                                                 ...        ...  
9874  {'지원동기': '인재와 기업이 함께 성장할 수 있는 곳, 그곳이 바로 웅진코웨이라...       True  
9875  {'현대자동차에 지원하게 된 동기 또는 해당 직무에 지원하게 된 동기는 무엇입니까?...       True  
9876  성격소개@@@대부분의 사람들이 평가하는 첫인상은 소심하고 내성적인 성격일것이라 생각...      False  
9877  [질문 1] 취업이라는 목표 이외에 인생의 큰 목표를 세운 경험, 그 목표를 실현하...      False  
9878  한전 자가소개서 양식 500자 12줄 이내@@@[질문 1] 취업이라는 목표 이외에 ...      False  

[9879 rows x 8 columns]

In [121]:
df.to_csv('G:\내 드라이브\web_crawling\linkareer_edit2.csv', mode='w', index=False)

In [122]:
df2 = pd.read_csv('G:\내 드라이브\web_crawling\linkareer_edit2.csv')

In [123]:
df2

UserInfos  \
0     한양대 ERICA / 기계공학과 / 학점 3.78/4.5 / 토익: 855, 토익스...   
1     지방사립 / 경제학 / 학점 4.15/4.5 / 토익: 720, 토익스피킹: 140...   
2     한서삼 / 컴퓨터공학과 / 학점 4.44 / 토익: 765, 토익스피킹: , 오픽:...   
3     인하대 / 컴퓨터공학과 / 학점 3.36 / 토익: , 토익스피킹: , 오픽: IM...   
4     경기대학교 / 응용통계학과 / 학점 3.8 / 토익: 705, 토익스피킹: , 오픽...   
...                                                 ...   
9874                 부산경상대학교 / 광고마케팅과 / 학점 3.8 / 토익 850   
9875  상명대학교 / 국제통상학 / 학점 4.3 / 토익 900, HSK 6급 / 해외인턴...   
9876                             지방전문대 / 공업화학 / 학점 3.06   
9877      지거국 / 전자공학 / 학점 3.7 / 토익 900대 / 음악봉사활동, 멘토링 등   
9878        지거국 / 전자공학 / 학점 3.7 / 토익 900 / 음악봉사활동, 멘토링등   

                                                Content  \
0     본인이 회사를 선택할때의 기준은 무엇이며, 왜 현대엔지니어링이 그 기준에 적합한지를...   
1     [금융 백종원] 백종원은 여러 매체를 통해 요리에 관한 각종 정보와 비결을 공유하며...   
2     현대엠엔소프트에 지원한 이유가 무엇인가요? (회사 및 직무에 대한 흥미와 커리어 계...   
3     본인이 이수한 SW 관련 전공 중, 가장 즐겁게 배웠던 수업 혹은 가장 좋은 성적을...   
4     1. 해당 지원분야 지원 동기와 해당 사업 참가 후 계획을 기술해주십시오(500자 ...   
...                                                 ...   
9874  1. 지원동기\n\n인재와 기업이 함께 성장할 수 있는 곳, 그곳이 바로 웅진코웨이...   
9875  1. 현대자동차에 지원하게 된 동기 또는 해당 직무에 지원하게 된 동기는 무엇입니까...   
9876  성격소개\n\n대부분의 사람들이 평가하는 첫인상은 소심하고 내성적인 성격일것이라 생...   
9877  [질문 1] 취업이라는 목표 이외에 인생의 큰 목표를 세운 경험, 그 목표를 실현하...   
9878  한전 자가소개서 양식 500자 12줄 이내\n\n[질문 1] 취업이라는 목표 이외에...   

                                                    URL  CompanyName  \
0     https://linkareer.com/cover-letter/25204?page=...      현대엔지니어링   
1     https://linkareer.com/cover-letter/25579?page=...         삼성생명   
2     https://linkareer.com/cover-letter/28660?page=...      현대엠엔소프트   
3     https://linkareer.com/cover-letter/28662?page=...      네이버파이낸셜   
4     https://linkareer.com/cover-letter/28755?page=...  한국지능정보사회지능원   
...                                                 ...          ...   
9874  https://linkareer.com/cover-letter/13306?page=...        웅진코웨이   
9875  https://linkareer.com/cover-letter/20654?page=...        현대자동차   
9876  https://linkareer.com/cover-letter/22028?page=...           심팩   
9877  https://linkareer.com/cover-letter/22089?page=...         한국전력   
9878  https://linkareer.com/cover-letter/17456?page=...         한국전력   

                       CompanyCareer CompanyDate  \
0                          자산관리 자산설비    2021 상반기   
1                             지점사무지원    2021 상반기   
2                       영상처리 및 컴퓨터비전    2021 상반기   
3     [NAVER FINANCIAL] 하계 SW개발자 인턴십    2021 상반기   
4                   데이터 분석 및 운영 청년인턴    2021 상반기   
...                              ...         ...   
9874                             마케팅    2013 하반기   
9875                            영업관리    2013 상반기   
9876                            용접관리    2013 상반기   
9877                        인턴[송,배전]    2013 상반기   
9878                        인턴[송,배전]    2013 상반기   

                                              NumUpdate  NotUpdate  
0     본인이 회사를 선택할때의 기준은 무엇이며, 왜 현대엔지니어링이 그 기준에 적합한지를...      False  
1     [금융 백종원] 백종원은 여러 매체를 통해 요리에 관한 각종 정보와 비결을 공유하며...      False  
2     현대엠엔소프트에 지원한 이유가 무엇인가요? (회사 및 직무에 대한 흥미와 커리어 계...      False  
3     본인이 이수한 SW 관련 전공 중, 가장 즐겁게 배웠던 수업 혹은 가장 좋은 성적을...      False  
4     {'해당 지원분야 지원 동기와 해당 사업 참가 후 계획을 기술해주십시오(500자 내...       True  
...                                                 ...        ...  
9874  {'지원동기': '인재와 기업이 함께 성장할 수 있는 곳, 그곳이 바로 웅진코웨이라...       True  
9875  {'현대자동차에 지원하게 된 동기 또는 해당 직무에 지원하게 된 동기는 무엇입니까?...       True  
9876  성격소개@@@대부분의 사람들이 평가하는 첫인상은 소심하고 내성적인 성격일것이라 생각...      False  
9877  [질문 1] 취업이라는 목표 이외에 인생의 큰 목표를 세운 경험, 그 목표를 실현하...      False  
9878  한전 자가소개서 양식 500자 12줄 이내@@@[질문 1] 취업이라는 목표 이외에 ...      False  

[9879 rows x 8 columns]